<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type"content="text/html;charset=UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Tracing Frontline</title>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css">
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.2.1/dist/jquery.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js"></script>
</head>
<body>
<div class="container">
<h3>Updated in 03-20-2020</h3>
<a href="archive/rss-03-20-2020.html" class="btn btn-primary btn-xs" role="button">Previous Week</a>
<a href="./rss-all.html" class="btn btn-primary btn-xs" role="button">Full Repertoire</a>
<h5><b>[Science]</b> <a href=http://science.sciencemag.org/cgi/content/short/367/6484/1336-f?rss=1>Antibody assembly in lampreys</a>. Williams, I. 2020-03-19T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAntibodyassemblyinlampreys">Abstract</button></h5><div id="abstractAntibodyassemblyinlampreys" class="collapse"></div>
<h5><b>[Science]</b> <a href=http://science.sciencemag.org/cgi/content/short/367/6484/1372?rss=1>Genetically targeted chemical assembly of functional materials in living cells, tissues, and animals</a>. Liu, J., Kim, Y. S., Richardson, C. E., Tom, A., R..., Pasca, S. P., Shen, K., Bao, Z., Deisseroth, K. 2020-03-19T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGeneticallytargetedchemicalassemblyoffunctionalmaterialsinlivingcellstissuesandanimals">Abstract</button></h5><div id="abstractGeneticallytargetedchemicalassemblyoffunctionalmaterialsinlivingcellstissuesandanimals" class="collapse"><p>The structural and functional complexity of multicellular biological systems, such as the brain, are beyond the reach of human design or assembly capabilities. Cells in living organisms may be recruited to construct synthetic materials or structures if treated as anatomically defined compartments for specific chemistry, harnessing biology for the assembly of complex functional structures. By integrating engineered-enzyme targeting and polymer chemistry, we genetically instructed specific living neurons to guide chemical synthesis of electrically functional (conductive or insulating) polymers at the plasma membrane. Electrophysiological and behavioral analyses confirmed that rationally designed, genetically targeted assembly of functional polymers not only preserved neuronal viability but also achieved remodeling of membrane properties and modulated cell type&ndash;specific behaviors in freely moving animals. This approach may enable the creation of diverse, complex, and functional structures and materials within living systems.</p></div>
<h5><b>[Science]</b> <a href=http://science.sciencemag.org/cgi/content/short/367/6484/eaay5012?rss=1>Insights into human genetic variation and population history from 929 diverse genomes</a>. Bergstro&#x0308;m, A., McCarthy, S. A., Hui, R., A.., Scally, A., Xue, Y., Durbin, R., Tyler-Smith, C. 2020-03-19T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractInsightsintohumangeneticvariationandpopulationhistoryfrom929diversegenomes">Abstract</button></h5><div id="abstractInsightsintohumangeneticvariationandpopulationhistoryfrom929diversegenomes" class="collapse"><p>Genome sequences from diverse human groups are needed to understand the structure of genetic variation in our species and the history of, and relationships between, different populations. We present 929 high-coverage genome sequences from 54 diverse human populations, 26 of which are physically phased using linked-read sequencing. Analyses of these genomes reveal an excess of previously undocumented common genetic variation private to southern Africa, central Africa, Oceania, and the Americas, but an absence of such variants fixed between major geographical regions. We also find deep and gradual population separations within Africa, contrasting population size histories between hunter-gatherer and agriculturalist groups in the past 10,000 years, and a contrast between single Neanderthal but multiple Denisovan source populations contributing to present-day human populations.</p></div>
<h5><b>[Science]</b> <a href=http://science.sciencemag.org/cgi/content/short/367/6484/eaay6690?rss=1>The genetic architecture of the human cerebral cortex</a>. Grasby, K. L., Jahanshad, N., Painter, J. N., Colo..aba, Cabrera, Grimmer, Khachaturian, Pausova, Mari 2020-03-19T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractThegeneticarchitectureofthehumancerebralcortex">Abstract</button></h5><div id="abstractThegeneticarchitectureofthehumancerebralcortex" class="collapse"><p>The cerebral cortex underlies our complex cognitive capabilities, yet little is known about the specific genetic loci that influence human cortical structure. To identify genetic variants that affect cortical structure, we conducted a genome-wide association meta-analysis of brain magnetic resonance imaging data from 51,665 individuals. We analyzed the surface area and average thickness of the whole cortex and 34 regions with known functional specializations. We identified 199 significant loci and found significant enrichment for loci influencing total surface area within regulatory elements that are active during prenatal cortical development, supporting the radial unit hypothesis. Loci that affect regional surface area cluster near genes in Wnt signaling pathways, which influence progenitor expansion and areal identity. Variation in cortical structure is genetically correlated with cognitive function, Parkinson&rsquo;s disease, insomnia, depression, neuroticism, and attention deficit hyperactivity disorder.</p></div><hr>
<h5><b>[NatBiotechnol]</b> <a href=https://www.nature.com/articles/s41587-020-0477-4>Publisher Correction: MEMOTE for standardized genome-scale metabolic model testing</a>. Cheng Zhang 2020-03-19 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPublisherCorrectionMEMOTEforstandardizedgenomescalemetabolicmodeltesting">Abstract</button></h5><div id="abstractPublisherCorrectionMEMOTEforstandardizedgenomescalemetabolicmodeltesting" class="collapse"><p>Nature Biotechnology, Published online: 19 March 2020; <a href="https://www.nature.com/articles/s41587-020-0477-4">doi:10.1038/s41587-020-0477-4</a></p>Publisher Correction: MEMOTE for standardized genome-scale metabolic model testing</div>
<h5><b>[NatBiotechnol]</b> <a href=https://www.nature.com/articles/s41587-020-0437-z>Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9–Cas12a platform</a>. Jason Moffat 2020-03-16 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGeneticinteractionmappingandexonresolutionfunctionalgenomicswithahybridCas9Cas12aplatform">Abstract</button></h5><div id="abstractGeneticinteractionmappingandexonresolutionfunctionalgenomicswithahybridCas9Cas12aplatform" class="collapse"><p>Nature Biotechnology, Published online: 16 March 2020; <a href="https://www.nature.com/articles/s41587-020-0437-z">doi:10.1038/s41587-020-0437-z</a></p>Combining Cas9 and Cas12a outperforms previous approaches for combinatorial screening in mammalian cells.</div>
<h5><b>[NatBiotechnol]</b> <a href=https://www.nature.com/articles/s41587-020-0455-x>Prime genome editing in rice and wheat</a>. Caixia Gao 2020-03-16 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPrimegenomeeditinginriceandwheat">Abstract</button></h5><div id="abstractPrimegenomeeditinginriceandwheat" class="collapse"><p>Nature Biotechnology, Published online: 16 March 2020; <a href="https://www.nature.com/articles/s41587-020-0455-x">doi:10.1038/s41587-020-0455-x</a></p>Prime editors are optimized for use in plants.</div><hr>
<h5><b>[NatPlants]</b> <a href=http://feeds.nature.com/~r/nplants/rss/current/~3/46F2N_fXVwc/s41477-020-0618-2><i>Anthoceros</i> genomes illuminate the origin of land plants and the unique biology of hornworts</a>. Péter Szövényi 2020-03-13 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractiAnthocerosigenomesilluminatetheoriginoflandplantsandtheuniquebiologyofhornworts">Abstract</button></h5><div id="abstractiAnthocerosigenomesilluminatetheoriginoflandplantsandtheuniquebiologyofhornworts" class="collapse"><p>Nature Plants, Published online: 13 March 2020; <a href="https://www.nature.com/articles/s41477-020-0618-2">doi:10.1038/s41477-020-0618-2</a></p>Analyses of three high-quality genomes of Anthoceros hornworts place hornworts as a sister clade to the lineage including liverworts and mosses, and provide insights into the unique biological features of hornworts.</div>
<h5><b>[NatPlants]</b> <a href=http://feeds.nature.com/~r/nplants/rss/current/~3/FJl3reF1ONc/s41477-020-0605-7>Integrated multi-omics framework of the plant response to jasmonic acid</a>. Joseph R. Ecker 2020-03-13 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractIntegratedmultiomicsframeworkoftheplantresponsetojasmonicacid">Abstract</button></h5><div id="abstractIntegratedmultiomicsframeworkoftheplantresponsetojasmonicacid" class="collapse"><p>Nature Plants, Published online: 13 March 2020; <a href="https://www.nature.com/articles/s41477-020-0605-7">doi:10.1038/s41477-020-0605-7</a></p>Five genome-wide approaches are integrated to give a comprehensive picture of jasmonic acid signalling networks in Arabidopsis. New genes, validated by mutant analysis, and important connections with other pathways are revealed.</div><hr>
<h5><b>[NatGenet]</b> <a href=https://www.nature.com/articles/s41588-020-0584-7>Interplay between whole-genome doubling and the accumulation of deleterious alterations in cancer evolution</a>. Nicholas McGranahan 2020-03-05 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractInterplaybetweenwholegenomedoublingandtheaccumulationofdeleteriousalterationsincancerevolution">Abstract</button></h5><div id="abstractInterplaybetweenwholegenomedoublingandtheaccumulationofdeleteriousalterationsincancerevolution" class="collapse"><p>Nature Genetics, Published online: 05 March 2020; <a href="https://www.nature.com/articles/s41588-020-0584-7">doi:10.1038/s41588-020-0584-7</a></p>Analysis of whole-genome doubling (WGD) by using cancer sequencing data combined with simulations of tumor evolution suggests that there is negative selection against homozygous loss of essential genes before WGD but not after.</div>
<h5><b>[NatGenet]</b> <a href=https://www.nature.com/articles/s41588-020-0598-1>Publisher Correction: Extrachromosomal circular DNA drives oncogenic genome remodeling in neuroblastoma</a>. Anton G. Henssen 2020-02-27 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPublisherCorrectionExtrachromosomalcircularDNAdrivesoncogenicgenomeremodelinginneuroblastoma">Abstract</button></h5><div id="abstractPublisherCorrectionExtrachromosomalcircularDNAdrivesoncogenicgenomeremodelinginneuroblastoma" class="collapse"><p>Nature Genetics, Published online: 27 February 2020; <a href="https://www.nature.com/articles/s41588-020-0598-1">doi:10.1038/s41588-020-0598-1</a></p>Publisher Correction: Extrachromosomal circular DNA drives oncogenic genome remodeling in neuroblastoma</div><hr>
<h5><b>[ScienceAdvances]</b> <a href=http://advances.sciencemag.org/cgi/content/short/6/12/eaay5696?rss=1>Recapitulating complex biological signaling environments using a multiplexed, DNA-patterning approach</a>. Scheideler, O. J., Yang, C., Kozminsky, M., Mosher.. A. W., Chern, S. A., Schaffer, D. V., Sohn, L. L. 2020-03-18T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractRecapitulatingcomplexbiologicalsignalingenvironmentsusingamultiplexedDNApatterningapproach">Abstract</button></h5><div id="abstractRecapitulatingcomplexbiologicalsignalingenvironmentsusingamultiplexedDNApatterningapproach" class="collapse"><p>Elucidating how the spatial organization of extrinsic signals modulates cell behavior and drives biological processes remains largely unexplored because of challenges in controlling spatial patterning of multiple microenvironmental cues in vitro. Here, we describe a high-throughput method that directs simultaneous assembly of multiple cell types and solid-phase ligands across length scales within minutes. Our method involves lithographically defining hierarchical patterns of unique DNA oligonucleotides to which complementary strands, attached to cells and ligands-of-interest, hybridize. Highlighting our method&rsquo;s power, we investigated how the spatial presentation of self-renewal ligand fibroblast growth factor-2 (FGF-2) and differentiation signal ephrin-B2 instruct single adult neural stem cell (NSC) fate. We found that NSCs have a strong spatial bias toward FGF-2 and identified an unexpected subpopulation exhibiting high neuronal differentiation despite spatially occupying patterned FGF-2 regions. Overall, our broadly applicable, DNA-directed approach enables mechanistic insight into how tissues encode regulatory information through the spatial presentation of heterogeneous signals.</p></div>
<h5><b>[ScienceAdvances]</b> <a href=http://advances.sciencemag.org/cgi/content/short/6/12/eaay6687?rss=1>Development of a CRISPR-SaCas9 system for projection- and function-specific gene editing in the rat brain</a>. Sun, H., Fu, S., Cui, S., Yin, X., Sun, X., Qi, X..., F.-Y., Liao, F.-F., Wang, X.-H., Yi, M., Wan, Y. 2020-03-18T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDevelopmentofaCRISPRSaCas9systemforprojectionandfunctionspecificgeneeditingintheratbrain">Abstract</button></h5><div id="abstractDevelopmentofaCRISPRSaCas9systemforprojectionandfunctionspecificgeneeditingintheratbrain" class="collapse"><p>A genome editing technique based on the clustered regularly interspaced short palindromic repeats (CRISPR)&ndash;associated endonuclease Cas9 enables efficient modification of genes in various cell types, including neurons. However, neuronal ensembles even in the same brain region are not anatomically or functionally uniform but divide into distinct subpopulations. Such heterogeneity requires gene editing in specific neuronal populations. We developed a CRISPR-SaCas9 system&ndash;based technique, and its combined application with anterograde/retrograde AAV vectors and activity-dependent cell-labeling techniques achieved projection- and function-specific gene editing in the rat brain. As a proof-of-principle application, we knocked down the <I>cbp</I> (CREB-binding protein), a sample target gene, in specific neuronal subpopulations in the medial prefrontal cortex, and demonstrated the significance of the projection- and function-specific CRISPR-SaCas9 system in revealing neuronal and circuit basis of memory. The high efficiency and specificity of our projection- and function-specific CRISPR-SaCas9 system could be widely applied in neural circuitry studies.</p></div><hr>
<h5><b>[NatRevGenet]</b> <a href=https://www.nature.com/articles/s41576-020-0229-9>Transcriptional profiling of physically interacting cells</a>. Linda Koch 2020-03-18 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTranscriptionalprofilingofphysicallyinteractingcells">Abstract</button></h5><div id="abstractTranscriptionalprofilingofphysicallyinteractingcells" class="collapse"><p>Nature Reviews Genetics, Published online: 18 March 2020; <a href="https://www.nature.com/articles/s41576-020-0229-9">doi:10.1038/s41576-020-0229-9</a></p>A new technology, named PIC-seq, combines cell sorting of physically interacting cells (PICs) with single-cell RNA sequencing and computational modelling to profile cellular interactions and their impact on gene expression.</div>
<h5><b>[NatRevGenet]</b> <a href=https://www.nature.com/articles/s41576-020-0218-z>Beyond broad strokes: sociocultural insights from the study of ancient genomes</a>. Carles Lalueza-Fox 2020-03-03 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractBeyondbroadstrokessocioculturalinsightsfromthestudyofancientgenomes">Abstract</button></h5><div id="abstractBeyondbroadstrokessocioculturalinsightsfromthestudyofancientgenomes" class="collapse"><p>Nature Reviews Genetics, Published online: 03 March 2020; <a href="https://www.nature.com/articles/s41576-020-0218-z">doi:10.1038/s41576-020-0218-z</a></p>With the increasing availability of ancient genomes from the same region or locality, the field of human history has started to address sociocultural aspects of human behaviour. The authors review recent case studies reconstructing the social consequences of past human migrations and advocate that this complex enterprise should be addressed by multidisciplinary teams.</div>
<h5><b>[NatRevGenet]</b> <a href=https://www.nature.com/articles/s41576-020-0216-1>Improving bee health through genomics</a>. Amro Zayed 2020-02-25 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractImprovingbeehealththroughgenomics">Abstract</button></h5><div id="abstractImprovingbeehealththroughgenomics" class="collapse"><p>Nature Reviews Genetics, Published online: 25 February 2020; <a href="https://www.nature.com/articles/s41576-020-0216-1">doi:10.1038/s41576-020-0216-1</a></p>Pollinators are critical for food security but bees, in particular, have experienced worrying population declines in recent years. This Review discusses how genomic technologies can be used to monitor and improve the health of managed and wild bee populations.</div>
<h5><b>[NatRevGenet]</b> <a href=https://www.nature.com/articles/s41576-020-0222-3>There and back again — ancient genes reveal early migrations</a>. Joseph Willson 2020-02-25 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractThereandbackagainancientgenesrevealearlymigrations">Abstract</button></h5><div id="abstractThereandbackagainancientgenesrevealearlymigrations" class="collapse"><p>Nature Reviews Genetics, Published online: 25 February 2020; <a href="https://www.nature.com/articles/s41576-020-0222-3">doi:10.1038/s41576-020-0222-3</a></p>A study in Cell using a novel technique for detecting Neanderthal genes in the genomes of modern humans reveals early gene flow and migratory events between ancient Africans and Europeans.</div>
<h5><b>[NatRevGenet]</b> <a href=https://www.nature.com/articles/s41576-020-0221-4>Transcriptomics of developmental fate</a>. Darren J. Burgess 2020-02-21 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTranscriptomicsofdevelopmentalfate">Abstract</button></h5><div id="abstractTranscriptomicsofdevelopmentalfate" class="collapse"><p>Nature Reviews Genetics, Published online: 21 February 2020; <a href="https://www.nature.com/articles/s41576-020-0221-4">doi:10.1038/s41576-020-0221-4</a></p>Two new studies in Science combine single-cell RNA sequencing with either lineage tracing or a computational framework to link transcriptomes to future developmental trajectories.</div><hr>
<h5><b>[Proc.Natl.Acad.Sci.U.S.A.]</b> <a href=http://feedproxy.google.com/~r/pnas/SMZM/~3/6xGdtbU2UgA/6145.short>Visualizing the translation and packaging of HIV-1 full-length RNA [Microbiology]</a>. Jianbo Chen, Yang Liu, Bin Wu, Olga A. Nikolaitchi.. R. Mohan, Jiji Chen, Vinay K. Pathak, Wei-Shau Hu 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractVisualizingthetranslationandpackagingofHIV1fulllengthRNAMicrobiology">Abstract</button></h5><div id="abstractVisualizingthetranslationandpackagingofHIV1fulllengthRNAMicrobiology" class="collapse">HIV-1 full-length RNA (HIV-1 RNA) plays a central role in viral replication, serving as a template for Gag/Gag-Pol translation and as a genome for the progeny virion. To gain a better understanding of the regulatory mechanisms of HIV-1 replication, we adapted a recently described system to visualize and track translation...</div>
<h5><b>[Proc.Natl.Acad.Sci.U.S.A.]</b> <a href=http://feedproxy.google.com/~r/pnas/SMZM/~3/AY6cIV86gU0/6264.short>Metabolic and genetic basis for auxotrophies in Gram-negative species [Systems Biology]</a>. Yara Seif, Kumari Sonal Choudhary, Ying Hefner, Amitesh Anand, Laurence Yang, Bernhard O. Palsson 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractMetabolicandgeneticbasisforauxotrophiesinGramnegativespeciesSystemsBiology">Abstract</button></h5><div id="abstractMetabolicandgeneticbasisforauxotrophiesinGramnegativespeciesSystemsBiology" class="collapse">Auxotrophies constrain the interactions of bacteria with their environment, but are often difficult to identify. Here, we develop an algorithm (AuxoFind) using genome-scale metabolic reconstruction to predict auxotrophies and apply it to a series of available genome sequences of over 1,300 Gram-negative strains. We identify 54 auxotrophs, along with the...</div>
<h5><b>[Proc.Natl.Acad.Sci.U.S.A.]</b> <a href=http://feedproxy.google.com/~r/pnas/SMZM/~3/hoLVEjpgg5U/5977.short>Integrated structural and evolutionary analysis reveals common mechanisms underlying adaptive evolution in mammals [Evolution]</a>. Greg Slodkowicz, Nick Goldman 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractIntegratedstructuralandevolutionaryanalysisrevealscommonmechanismsunderlyingadaptiveevolutioninmammalsEvolution">Abstract</button></h5><div id="abstractIntegratedstructuralandevolutionaryanalysisrevealscommonmechanismsunderlyingadaptiveevolutioninmammalsEvolution" class="collapse">Understanding the molecular basis of adaptation to the environment is a central question in evolutionary biology, yet linking detected signatures of positive selection to molecular mechanisms remains challenging. Here we demonstrate that combining sequence-based phylogenetic methods with structural information assists in making such mechanistic interpretations on a genomic scale. Our...</div>
<h5><b>[Proc.Natl.Acad.Sci.U.S.A.]</b> <a href=http://feedproxy.google.com/~r/pnas/SMZM/~3/YgMY6-7hy8A/6121.short>Cell-penetrating peptide inhibits retromer-mediated human papillomavirus trafficking during virus entry [Microbiology]</a>. Pengwei Zhang, Ruben Moreno, Paul F. Lambert, Daniel DiMaio 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCellpenetratingpeptideinhibitsretromermediatedhumanpapillomavirustraffickingduringvirusentryMicrobiology">Abstract</button></h5><div id="abstractCellpenetratingpeptideinhibitsretromermediatedhumanpapillomavirustraffickingduringvirusentryMicrobiology" class="collapse">Virus replication requires critical interactions between viral proteins and cellular proteins that mediate many aspects of infection, including the transport of viral genomes to the site of replication. In human papillomavirus (HPV) infection, the cellular protein complex known as retromer binds to the L2 capsid protein and sorts incoming virions...</div>
<h5><b>[Proc.Natl.Acad.Sci.U.S.A.]</b> <a href=http://feedproxy.google.com/~r/pnas/SMZM/~3/zth5bqVQkzA/5617.short>Programming colloidal bonding using DNA strand-displacement circuitry [Chemistry]</a>. Xiang Zhou, Dongbao Yao, Wenqiang Hua, Ningdong Hu..ijun Guo, Shiyan Xiao, Fenggang Bian, Haojun Liang 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractProgrammingcolloidalbondingusingDNAstranddisplacementcircuitryChemistry">Abstract</button></h5><div id="abstractProgrammingcolloidalbondingusingDNAstranddisplacementcircuitryChemistry" class="collapse">As a strategy for regulating entropy, thermal annealing is a commonly adopted approach for controlling dynamic pathways in colloid assembly. By coupling DNA strand-displacement circuits with DNA-functionalized colloid assembly, we developed an enthalpy-mediated strategy for achieving the same goal while working at a constant temperature. Using this tractable approach allows...</div><hr>
<h5><b>[MolPlant]</b> <a href=https://www.cell.com/molecular-plant/fulltext/S1674-2052(20)30066-6?rss=yes>Bacterial effectors induce oligomerization of immune receptor ZAR1 in vivo</a>. Meijuan Hu, Jinfeng Qi, Guozhi Bi, Jian-Min Zhou 2020-03-16 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractBacterialeffectorsinduceoligomerizationofimmunereceptorZAR1invivo">Abstract</button></h5><div id="abstractBacterialeffectorsinduceoligomerizationofimmunereceptorZAR1invivo" class="collapse">The bacterial pathogen effectors AvrAC and HopZ1a induce oligomerization of the Arabidopsis NLR protein ZAR1 in protoplasts. Structural requirements for ZAR1 resistosome assembly in vitro are also essential for HopZ1a-induced ZAR1 oligomerization in vivo and disease resistance in plants, providing evidence that the ZAR1 resistosome forms in vivo during immune activation.</div>
<h5><b>[MolPlant]</b> <a href=https://www.cell.com/molecular-plant/fulltext/S1674-2052(20)30030-7?rss=yes>A Domestication-Associated Gene GmPRR3b Regulates the Circadian Clock and Flowering Time in Soybean</a>. Cong Li, Ying-hui Li, Yanfei Li, Hongfeng Lu, Huil..Xinan Zhou, Scott A. Jackson, Bin Liu, Li-juan Qiu 2020-01-31 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractADomesticationAssociatedGeneGmPRR3bRegulatestheCircadianClockandFloweringTimeinSoybean">Abstract</button></h5><div id="abstractADomesticationAssociatedGeneGmPRR3bRegulatestheCircadianClockandFloweringTimeinSoybean" class="collapse">Natural variation in flowering time is critical for geographic adaptation of soybean. A circadian clock gene, GmPRR3b, was identified as a major flowering time QTL through a genome-wide association study. GmPRR3b haplotype 6 is identified to be associated with early flowering and vigorous growth traits, which has been extensively utilized during domestication and breeding of modern cultivars.</div><hr>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30196-2?rss=yes>Critical Roles of a RhoGEF-Anillin Module in Septin Architectural Remodeling during Cytokinesis</a>. Xi Chen, Kangji Wang, Tatyana Svitkina, Erfei Bi 2020-03-19 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCriticalRolesofaRhoGEFAnillinModuleinSeptinArchitecturalRemodelingduringCytokinesis">Abstract</button></h5><div id="abstractCriticalRolesofaRhoGEFAnillinModuleinSeptinArchitecturalRemodelingduringCytokinesis" class="collapse">It remains unknown how septins are remodeled from an hourglass-shaped structure to a double ring during cytokinesis. Chen et al. show that the RhoGEF Bud3 and the anillin-like protein Bud4 in budding yeast play distinct and essential roles in the assembly of a zonal transitional septin hourglass and its subsequent remodeling into a double ring.</div>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30189-5?rss=yes>Marine Sediments Illuminate Chlamydiae Diversity and Evolution</a>. Jennah E. Dharamshi, Daniel Tamarit, Laura Eme, Co..teffen L. Jørgensen, Anja Spang, Thijs J.G. Ettema 2020-03-05 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractMarineSedimentsIlluminateChlamydiaeDiversityandEvolution">Abstract</button></h5><div id="abstractMarineSedimentsIlluminateChlamydiaeDiversityandEvolution" class="collapse">Dharamshi et al. find abundant, diverse, and active Chlamydiae in deep anoxic marine sediments. Using metagenomics, chlamydial genomes are obtained that form several new clades. Analyses of these genomes provide new insights into the evolution and host association of the Chlamydiae phylum, indicating that some are not symbionts of eukaryotic hosts.</div>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30027-0?rss=yes>Epithelial Planar Bipolarity Emerges from Notch-Mediated Asymmetric Inhibition of Emx2</a>. Eva L. Kozak, Subarna Palit, Jerónimo R. Miranda-R..lfgang Enard, Fabian J. Theis, Hernán López-Schier 2020-02-27 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractEpithelialPlanarBipolarityEmergesfromNotchMediatedAsymmetricInhibitionofEmx2">Abstract</button></h5><div id="abstractEpithelialPlanarBipolarityEmergesfromNotchMediatedAsymmetricInhibitionofEmx2" class="collapse">Kozak et al. combine single-cell RNA sequencing, pseudotime trajectory analysis, and mutagenesis to reveal the initial symmetry-breaking step that results in the establishment of planar bipolarity in a sensory epithelium.</div>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30118-4?rss=yes>The Genome of Caenorhabditis bovis</a>. Lewis Stevens, Stefan Rooke, Laura C. Falzon, Euni.. Ogendo, Joseph Ogola, Eric M. Fèvre, Mark Blaxter 2020-02-27 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTheGenomeofCaenorhabditisbovis">Abstract</button></h5><div id="abstractTheGenomeofCaenorhabditisbovis" class="collapse">Stevens et al. isolate Caenorhabditis bovis from the ear of a cow in Kenya and sequence its genome in a nearby field laboratory using the Oxford Nanopore MinION. They compare the C. bovis genome with that of C. elegans, identify genes that may relate to its unusual ecology, and determine how C. bovis is related to other Caenorhabditis species.</div>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30105-6?rss=yes>Extreme Genome and Nervous System Streamlining in the Invertebrate Parasite Intoshia variabili</a>. George S. Slyusarev, Viktor V. Starunov, Anton S. ..ndarenko, Natalia A. Zorina, Natalya I. Bondarenko 2020-02-20 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractExtremeGenomeandNervousSystemStreamliningintheInvertebrateParasiteIntoshiavariabili">Abstract</button></h5><div id="abstractExtremeGenomeandNervousSystemStreamliningintheInvertebrateParasiteIntoshiavariabili" class="collapse">Slyusarev et al. report on the morphology, genome structure, and content of the smallest Orthonectida species Intoshia variabili. This orthonectid with an extremely simplified nervous system has the smallest known metazoan genome, with one of the fewest numbers of genes reported.</div>
<h5><b>[CurrBiol]</b> <a href=https://www.cell.com/current-biology/fulltext/S0960-9822(20)30093-2?rss=yes>Single-Cell Transcriptomes Reveal Diverse Regulatory Strategies for Olfactory Receptor Expression and Axon Targeting</a>. Hongjie Li, Tongchao Li, Felix Horns, Jiefu Li, Qi..e, David J. Luginbuhl, Stephen R. Quake, Liqun Luo 2020-02-13 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractSingleCellTranscriptomesRevealDiverseRegulatoryStrategiesforOlfactoryReceptorExpressionandAxonTargeting">Abstract</button></h5><div id="abstractSingleCellTranscriptomesRevealDiverseRegulatoryStrategiesforOlfactoryReceptorExpressionandAxonTargeting" class="collapse">In this study, Li et al. perform single-cell RNA sequencing of developing olfactory receptor neurons (ORNs) in Drosophila and reveal that ORNs utilize diverse transcriptional strategies to coordinate their olfactory receptor expression and axon targeting.</div><hr>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa062/5810090?rss=1>Tbx2a modulates switching of RH2 and LWS opsin gene expression</a>. Sandkam B, Campello L, O’Brien C, et al. Thu, 19 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTbx2amodulatesswitchingofRH2andLWSopsingeneexpression">Abstract</button></h5><div id="abstractTbx2amodulatesswitchingofRH2andLWSopsingeneexpression" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Sensory systems are tuned by selection to maximize organismal fitness in particular environments. This tuning has implications for intraspecies communication, the maintenance of species boundaries, and speciation. Tuning of color vision largely depends on the sequence of the expressed opsin proteins. To improve tuning of visual sensitivities to shifts in habitat or foraging ecology over the course of development, many organisms change which opsins are expressed. Changes in this developmental sequence (heterochronic shifts) can create differences in visual sensitivity among closely related species. The genetic mechanisms by which these developmental shifts occur are poorly understood. Here, we use QTL analyses, genome sequencing, and gene expression studies in African cichlid fishes to identify a role for the transcription factor Tbx2a in driving a switch between <span style="font-style:italic;">LWS</span> and <span style="font-style:italic;">RH2</span> opsin expression. We identify binding sites for Tbx2a in the <span style="font-style:italic;">LWS</span> promoter and the highly conserved Locus Control Region of <span style="font-style:italic;">RH2</span> which concurrently promote <span style="font-style:italic;">LWS</span> expression while repressing <span style="font-style:italic;">RH2</span> expression. We also present evidence that a single change in <span style="font-style:italic;">Tbx2a</span> regulatory sequence has led to a species difference in visual tuning, providing the first mechanistic model for the evolution of rapid switches in sensory tuning. This difference in visual tuning likely has important roles in evolution as it corresponds to differences in diet, microhabitat choice, and male nuptial coloration.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa077/5810091?rss=1>Unmatched level of molecular convergence among deeply divergent complex multicellular fungi</a>. MerĂŠnyi Z, Prasanna A, Wang Z, et al. Thu, 19 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractUnmatchedlevelofmolecularconvergenceamongdeeplydivergentcomplexmulticellularfungi">Abstract</button></h5><div id="abstractUnmatchedlevelofmolecularconvergenceamongdeeplydivergentcomplexmulticellularfungi" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Convergent evolution is pervasive in nature, but it is poorly understood how various constraints and natural selection limit the diversity of evolvable phenotypes. Here, we analyze the transcriptome across fruiting body development to understand the independent evolution of complex multicellularity in the two largest clades of fungi—the Agarico- and Pezizomycotina. Despite &gt;650 million years of divergence between these clades, we find that very similar sets of genes have convergently been co-opted for complex multicellularity, followed by expansions of their gene families by duplications. Over 82% of shared multicellularity-related gene families were expanding in both clades, indicating a high prevalence of convergence also at the gene-family level. This convergence is coupled with a rich inferred repertoire of multicellularity-related genes in the most recent common ancestor of the Agarico- and Pezizomycotina, consistent with the hypothesis that the coding capacity of ancestral fungal genomes might have promoted the repeated evolution of complex multicellularity. We interpret this repertoire as an indication of evolutionary predisposition of fungal ancestors for evolving complex multicellular fruiting bodies. Our work suggests that evolutionary convergence may happen not only when organisms are closely related or are under similar selection pressures, but also when ancestral genomic repertoires render certain evolutionary trajectories more likely than others, even across large phylogenetic distances.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa078/5810089?rss=1>Dynamic evolution of euchromatic satellites on the X chromosome in Drosophila melanogaster and the simulans clade</a>. Sproul J, Khost D, Eickbush D, et al. Thu, 19 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDynamicevolutionofeuchromaticsatellitesontheXchromosomeinDrosophilamelanogasterandthesimulansclade">Abstract</button></h5><div id="abstractDynamicevolutionofeuchromaticsatellitesontheXchromosomeinDrosophilamelanogasterandthesimulansclade" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Satellite DNAs (satDNAs) are among the most dynamically evolving components of eukaryotic genomes and play important roles in genome regulation, genome evolution, and speciation. Despite their abundance and functional impact, we know little about the evolutionary dynamics and molecular mechanisms that shape satDNA distributions in genomes. Here we use high-quality genome assemblies to study the evolutionary dynamics of two complex satDNAs, <span style="font-style:italic;">Rsp-like</span> and <span style="font-style:italic;">1.688</span> gm/cm<sup>3</sup>, in <span style="font-style:italic;">Drosophila melanogaster</span> and its three nearest relatives in the <span style="font-style:italic;">simulans</span> clade. We show that large blocks of these repeats are highly dynamic in the heterochromatin, where their genomic location varies across species. We discovered that small blocks of satDNA that are abundant in X chromosome euchromatin are similarly dynamic, with repeats changing in abundance, location, and composition among species. We detail the proliferation of a rare satellite (<span style="font-style:italic;">Rsp-like</span>) across the X chromosome in <span style="font-style:italic;">D. simulans</span> and <span style="font-style:italic;">D. mauritiana. Rsp-like</span> spread by inserting into existing clusters of the older, more abundant <span style="font-style:italic;">1.688</span> satellite, in events likely facilitated by microhomology-mediated repair pathways. We show that <span style="font-style:italic;">Rsp-like</span> is abundant on extrachromosomal circular DNA in <span style="font-style:italic;">D. simulans</span>, which may have contributed to its dynamic evolution. Intralocus satDNA expansions via unequal exchange and the movement of higher-order repeats also contribute to the fluidity of the repeat landscape. We find evidence that euchromatic satDNA repeats experience cycles of proliferation and diversification somewhat analogous to bursts of transposable element proliferation. Our study lays a foundation for mechanistic studies of satDNA proliferation and the functional and evolutionary consequences of satDNA movement.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa076/5809148?rss=1>Genomic signature of shifts in selection in a sub-alpine ant and its physiological adaptations</a>. Cicconardi F, Krapf P, D'Annessa I, et al. Tue, 17 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGenomicsignatureofshiftsinselectioninasubalpineantanditsphysiologicaladaptations">Abstract</button></h5><div id="abstractGenomicsignatureofshiftsinselectioninasubalpineantanditsphysiologicaladaptations" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Understanding how organisms adapt to extreme environments is fundamental and can provide insightful case studies for both evolutionary biology and climate-change biology. Here, we take advantage of the vast diversity of lifestyles in ants to identify genomic signatures of adaptation to extreme habitats such as high altitude. We hypothesised two parallel patterns would occur in a genome adapting to an extreme habitat: <span style="font-style:italic;">i</span>) strong positive selection on genes related to adaptation and, <span style="font-style:italic;">ii</span>) a relaxation of previous purifying selection. We tested this hypothesis by sequencing the high-elevation specialist <span style="font-style:italic;">Tetramorium alpestre</span> and four other phylogenetic related species. In support of our hypothesis, we recorded a strong shift of selective forces in <span style="font-style:italic;">T. alpestre</span>, in particular a stronger magnitude of diversifying and relaxed selection when compared to all other ants. We further disentangled candidate molecular adaptations in both gene expression and protein-coding sequence that were identified by our genome-wide analyses. In particular, we demonstrate that <span style="font-style:italic;">T. alpestre</span> has <span style="font-style:italic;">i</span>) a higher level of expression for <span style="font-style:italic;">stv</span> and other heat-shock proteins in chill shock tests, and <span style="font-style:italic;">ii</span>) enzymatic enhancement of Hex-T1, a rate-limiting regulatory enzyme that controls the entry of glucose into the glycolytic pathway. Together, our analyses highlight the adaptive molecular changes that support colonisation of high-altitude environments.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/933/5652084?rss=1>Estimating Pangenomes with Roary</a>. Sitto F, Battistuzzi F, Hall B. Tue, 17 Dec 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractEstimatingPangenomeswithRoary">Abstract</button></h5><div id="abstractEstimatingPangenomeswithRoary" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>A description of the genetic makeup of a species based on a single genome is often insufficient because it ignores the variability in gene repertoire among multiple strains. The estimation of the pangenome of a species is a solution to this issue as it provides an overview of genes that are shared by all strains and genes that are present in only some of the genomes. These different sets of genes can then be analyzed functionally to explore correlations with unique phenotypes and adaptations. This protocol presents the usage of Roary, a Linux-native pangenome application. Roary is a straightforward software that provides 1) an overview about core and accessory genes for those interested in general trends and, also, 2) detailed information on gene presence/absence in each genome for in-depth analyses. Results are provided both in text and graphic format.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/849/5626244?rss=1>Early Stage Adaptation of a Mesophilic Green Alga to Antarctica: Systematic Increases in Abundance of Enzymes and LEA Proteins</a>. Wang Y, Liu X, Gao H, et al. Wed, 20 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractEarlyStageAdaptationofaMesophilicGreenAlgatoAntarcticaSystematicIncreasesinAbundanceofEnzymesandLEAProteins">Abstract</button></h5><div id="abstractEarlyStageAdaptationofaMesophilicGreenAlgatoAntarcticaSystematicIncreasesinAbundanceofEnzymesandLEAProteins" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>It is known that adaptive evolution in permanently cold environments drives cold adaptation in enzymes. However, how the relatively high enzyme activities were achieved in cold environments prior to cold adaptation of enzymes is unclear. Here we report that an Antarctic strain of <span style="font-style:italic;">Chlorella vulgaris</span>, called NJ-7, acquired the capability to grow at near 0 °C temperatures and greatly enhanced freezing tolerance after systematic increases in abundance of enzymes/proteins and positive selection of certain genes. Having diverged from the temperate strain UTEX259 of the same species 2.5 (1.1–4.1) to 2.6 (1.0–4.5) Ma, NJ-7 retained the basic mesophilic characteristics and genome structures. Nitrate reductases in the two strains are highly similar in amino acid sequence and optimal temperature, but the NJ-7 one showed significantly higher abundance and activity. Quantitative proteomic analyses indicated that several cryoprotective proteins (LEA), many enzymes involved in carbon metabolism and a large number of other enzymes/proteins, were more abundant in NJ-7 than in UTEX259. Like nitrate reductase, most of these enzymes were not upregulated in response to cold stress. Thus, compensation of low specific activities by increased enzyme abundance appears to be an important strategy for early stage cold adaptation to Antarctica, but such enzymes are mostly not involved in cold acclimation upon transfer from favorable temperatures to near 0 °C temperatures.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/627/5626191?rss=1>Recurrent Collection of Drosophila melanogaster from Wild African Environments and Genomic Insights into Species History</a>. Sprengelmeyer Q, Mansourian S, Lange J, et al. Fri, 15 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractRecurrentCollectionofDrosophilamelanogasterfromWildAfricanEnvironmentsandGenomicInsightsintoSpeciesHistory">Abstract</button></h5><div id="abstractRecurrentCollectionofDrosophilamelanogasterfromWildAfricanEnvironmentsandGenomicInsightsintoSpeciesHistory" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>A long-standing enigma concerns the geographic and ecological origins of the intensively studied vinegar fly, <span style="font-style:italic;">Drosophila melanogaster</span>. This globally distributed human commensal is thought to originate from sub-Saharan Africa, yet until recently, it had never been reported from undisturbed wilderness environments that could reflect its precommensal niche. Here, we document the collection of 288 <span style="font-style:italic;">D. melanogaster</span> individuals from multiple African wilderness areas in Zambia, Zimbabwe, and Namibia. The presence of <span style="font-style:italic;">D. melanogaster</span> in these remote woodland environments is consistent with an ancestral range in southern-central Africa, as opposed to equatorial regions. After sequencing the genomes of 17 wilderness-collected flies collected from Kafue National Park in Zambia, we found reduced genetic diversity relative to town populations, elevated chromosomal inversion frequencies, and strong differences at specific genes including known insecticide targets. Combining these genomes with existing data, we probed the history of this species’ geographic expansion. Demographic estimates indicated that expansion from southern-central Africa began ∼10,000 years ago, with a Saharan crossing soon after, but expansion from the Middle East into Europe did not begin until roughly 1,400 years ago. This improved model of demographic history will provide an important resource for future evolutionary and genomic studies of this key model organism. Our findings add context to the history of <span style="font-style:italic;">D. melanogaster</span>, while opening the door for future studies on the biological basis of adaptation to human environments.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/839/5626190?rss=1>Domestication of High-Copy Transposons Underlays the Wheat Small RNA Response to an Obligate Pathogen</a>. Poretti M, Praz C, Meile L, et al. Fri, 15 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDomesticationofHighCopyTransposonsUnderlaystheWheatSmallRNAResponsetoanObligatePathogen">Abstract</button></h5><div id="abstractDomesticationofHighCopyTransposonsUnderlaystheWheatSmallRNAResponsetoanObligatePathogen" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Plant genomes have evolved several evolutionary mechanisms to tolerate and make use of transposable elements (TEs). Of these, transposon domestication into cis-regulatory and microRNA (miRNA) sequences is proposed to contribute to abiotic/biotic stress adaptation in plants. The wheat genome is derived at 85% from TEs, and contains thousands of miniature inverted-repeat transposable elements (MITEs), whose sequences are particularly prone for domestication into miRNA precursors. In this study, we investigate the contribution of TEs to the wheat small RNA immune response to the lineage-specific, obligate powdery mildew pathogen. We show that MITEs of the <span style="font-style:italic;">Mariner</span> superfamily contribute the largest diversity of miRNAs to the wheat immune response. In particular, MITE precursors of miRNAs are wide-spread over the wheat genome, and highly conserved copies are found in the Lr34 and QPm.tut-4A mildew resistance loci. Our work suggests that transposon domestication is an important evolutionary force driving miRNA functional innovation in wheat immunity.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/923/5614437?rss=1>Unbiased Estimation of Linkage Disequilibrium from Unphased Data</a>. Ragsdale A, Gravel S, Kim Y. Thu, 14 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractUnbiasedEstimationofLinkageDisequilibriumfromUnphasedData">Abstract</button></h5><div id="abstractUnbiasedEstimationofLinkageDisequilibriumfromUnphasedData" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Linkage disequilibrium (LD) is used to infer evolutionary history, to identify genomic regions under selection, and to dissect the relationship between genotype and phenotype. In each case, we require accurate estimates of LD statistics from sequencing data. Unphased data present a challenge because multilocus haplotypes cannot be inferred exactly. Widely used estimators for the common statistics <span style="font-style:italic;">r</span><sup>2</sup> and <span style="font-style:italic;">D</span><sup>2</sup> exhibit large and variable upward biases that complicate interpretation and comparison across cohorts. Here, we show how to find unbiased estimators for a wide range of two-locus statistics, including <span style="font-style:italic;">D</span><sup>2</sup>, for both single and multiple randomly mating populations. These unbiased statistics are particularly well suited to estimate effective population sizes from unlinked loci in small populations. We develop a simple inference pipeline and use it to refine estimates of recent effective population sizes of the threatened Channel Island Fox populations.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/828/5625159?rss=1>Somatic Mutations and Genome Stability Maintenance in Clonal Coral Colonies</a>. López E, Palumbi S, Yeager M. Wed, 13 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractSomaticMutationsandGenomeStabilityMaintenanceinClonalCoralColonies">Abstract</button></h5><div id="abstractSomaticMutationsandGenomeStabilityMaintenanceinClonalCoralColonies" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>One challenge for multicellular organisms is maintaining genome stability in the face of mutagens across long life spans. Imperfect genome maintenance leads to mutation accumulation in somatic cells, which is associated with tumors and senescence in vertebrates. Colonial reef-building corals are often large, can live for hundreds of years, rarely develop recognizable tumors, and are thought to convert somatic cells into gamete producers, so they are a pivotal group in which to understand long-term genome maintenance. To measure rates and patterns of somatic mutations, we analyzed transcriptomes from 17 to 22 branches from each of four <span style="font-style:italic;">Acropora hyacinthus</span> colonies, determined putative single nucleotide variants, and verified them with Sanger resequencing. Unlike for human skin carcinomas, there is no signature of mutations caused by UV damage, indicating either higher efficiency of repair than in vertebrates, or strong sunscreen protection in these shallow water tropical animals. The somatic mutation frequency per nucleotide in <span style="font-style:italic;">A. hyacinthus</span> is on the same order of magnitude (10<sup>−7</sup>) as noncancerous human somatic cells, and accumulation of mutations with age is similar. Loss of heterozygosity variants outnumber gain of heterozygosity mutations ∼2:1. Although the mutation frequency is similar in mammals and corals, the preponderance of loss of heterozygosity changes and potential selection may reduce the frequency of deleterious mutations in colonial animals like corals. This may limit the deleterious effects of somatic mutations on the coral organism as well as potential offspring.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/799/5618730?rss=1>Developmental Systems Drift and the Drivers of Sex Chromosome Evolution</a>. Cauret C, Gansauge M, Tupper A, et al. Mon, 11 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDevelopmentalSystemsDriftandtheDriversofSexChromosomeEvolution">Abstract</button></h5><div id="abstractDevelopmentalSystemsDriftandtheDriversofSexChromosomeEvolution" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Phenotypic invariance—the outcome of purifying selection—is a hallmark of biological importance. However, invariant phenotypes might be controlled by diverged genetic systems in different species. Here, we explore how an important and invariant phenotype—the development of sexually differentiated individuals—is controlled in over two dozen species in the frog family Pipidae. We uncovered evidence in different species for 1) an ancestral W chromosome that is not found in many females and is found in some males, 2) independent losses and 3) autosomal segregation of this W chromosome, 4) changes in male versus female heterogamy, and 5) substantial variation among species in recombination suppression on sex chromosomes. We further provide evidence of, and evolutionary context for, the origins of at least seven distinct systems for regulating sex determination among three closely related genera. These systems are distinct in their genomic locations, evolutionary origins, and/or male versus female heterogamy. Our findings demonstrate that the developmental control of sexual differentiation changed via loss, sidelining, and empowerment of a mechanistically influential gene, and offer insights into novel factors that impinge on the diverse evolutionary fates of sex chromosomes.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/611/5618728?rss=1>Ancient DNA Reconstructs the Genetic Legacies of Precontact Puerto Rico Communities</a>. Nieves-Colón M, Pestle W, Reynolds A, et al. Sat, 09 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAncientDNAReconstructstheGeneticLegaciesofPrecontactPuertoRicoCommunities">Abstract</button></h5><div id="abstractAncientDNAReconstructstheGeneticLegaciesofPrecontactPuertoRicoCommunities" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Indigenous peoples have occupied the island of Puerto Rico since at least 3000 BC. Due to the demographic shifts that occurred after European contact, the origin(s) of these ancient populations, and their genetic relationship to present-day islanders, are unclear. We use ancient DNA to characterize the population history and genetic legacies of precontact Indigenous communities from Puerto Rico. Bone, tooth, and dental calculus samples were collected from 124 individuals from three precontact archaeological sites: Tibes, Punta Candelero, and Paso del Indio. Despite poor DNA preservation, we used target enrichment and high-throughput sequencing to obtain complete mitochondrial genomes (mtDNA) from 45 individuals and autosomal genotypes from two individuals. We found a high proportion of Native American mtDNA haplogroups A2 and C1 in the precontact Puerto Rico sample (40% and 44%, respectively). This distribution, as well as the haplotypes represented, supports a primarily Amazonian South American origin for these populations and mirrors the Native American mtDNA diversity patterns found in present-day islanders. Three mtDNA haplotypes from precontact Puerto Rico persist among Puerto Ricans and other Caribbean islanders, indicating that present-day populations are reservoirs of precontact mtDNA diversity. Lastly, we find similarity in autosomal ancestry patterns between precontact individuals from Puerto Rico and the Bahamas, suggesting a shared component of Indigenous Caribbean ancestry with close affinity to South American populations. Our findings contribute to a more complete reconstruction of precontact Caribbean population history and explore the role of Indigenous peoples in shaping the biocultural diversity of present-day Puerto Ricans and other Caribbean islanders.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/904/5618729?rss=1>Optimizing Phylogenomics with Rapidly Evolving Long Exons: Comparison with Anchored Hybrid Enrichment and Ultraconserved Elements</a>. Karin B, Gamble T, Jackman T, et al. Sat, 09 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractOptimizingPhylogenomicswithRapidlyEvolvingLongExonsComparisonwithAnchoredHybridEnrichmentandUltraconservedElements">Abstract</button></h5><div id="abstractOptimizingPhylogenomicswithRapidlyEvolvingLongExonsComparisonwithAnchoredHybridEnrichmentandUltraconservedElements" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Marker selection has emerged as an important component of phylogenomic study design due to rising concerns of the effects of gene tree estimation error, model misspecification, and data-type differences. Researchers must balance various trade-offs associated with locus length and evolutionary rate among other factors. The most commonly used reduced representation data sets for phylogenomics are ultraconserved elements (UCEs) and Anchored Hybrid Enrichment (AHE). Here, we introduce Rapidly Evolving Long Exon Capture (RELEC), a new set of loci that targets single exons that are both rapidly evolving (evolutionary rate faster than <span style="font-style:italic;">RAG1</span>) and relatively long in length (&gt;1,500 bp), while at the same time avoiding paralogy issues across amniotes. We compare the RELEC data set to UCEs and AHE in squamate reptiles by aligning and analyzing orthologous sequences from 17 squamate genomes, composed of 10 snakes and 7 lizards. The RELEC data set (179 loci) outperforms AHE and UCEs by maximizing per-locus genetic variation while maintaining presence and orthology across a range of evolutionary scales. RELEC markers show higher phylogenetic informativeness than UCE and AHE loci, and RELEC gene trees show greater similarity to the species tree than AHE or UCE gene trees. Furthermore, with fewer loci, RELEC remains computationally tractable for full Bayesian coalescent species tree analyses. We contrast RELEC to and discuss important aspects of comparable methods, and demonstrate how RELEC may be the most effective set of loci for resolving difficult nodes and rapid radiations. We provide several resources for capturing or extracting RELEC loci from other amniote groups.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/730/5614851?rss=1>Phylogenomics Identifies an Ancestral Burst of Gene Duplications Predating the Diversification of Aphidomorpha</a>. Julca I, Marcet-Houben M, Cruz F, et al. Fri, 08 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPhylogenomicsIdentifiesanAncestralBurstofGeneDuplicationsPredatingtheDiversificationofAphidomorpha">Abstract</button></h5><div id="abstractPhylogenomicsIdentifiesanAncestralBurstofGeneDuplicationsPredatingtheDiversificationofAphidomorpha" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Aphids (Aphidoidea) are a diverse group of hemipteran insects that feed on plant phloem sap. A common finding in studies of aphid genomes is the presence of a large number of duplicated genes. However, when these duplications occurred remains unclear, partly due to the high relatedness of sequenced species. To better understand the origin of aphid duplications we sequenced and assembled the genome of <span style="font-style:italic;">Cinara cedri</span>, an early branching lineage (Lachninae) of the Aphididae family. We performed a phylogenomic comparison of this genome with 20 other sequenced genomes, including the available genomes of five other aphids, along with the transcriptomes of two species belonging to Adelgidae (a closely related clade to the aphids) and Coccoidea. We found that gene duplication has been pervasive throughout the evolution of aphids, including many parallel waves of recent, species-specific duplications. Most notably, we identified a consistent set of very ancestral duplications, originating from a large-scale gene duplication predating the diversification of Aphidomorpha (comprising aphids, phylloxerids, and adelgids). Genes duplicated in this ancestral wave are enriched in functions related to traits shared by Aphidomorpha, such as association with endosymbionts, and adaptation to plant defenses and phloem-sap-based diet. The ancestral nature of this duplication wave (106–227 Ma) and the lack of sufficiently conserved synteny make it difficult to conclude whether it originated from a whole-genome duplication event or, alternatively, from a burst of large-scale segmental duplications. Genome sequencing of other aphid species belonging to different Aphidomorpha and related lineages may clarify these findings.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/757/5614436?rss=1>A Burst of Genetic Innovation in Drosophila Actin-Related Proteins for Testis-Specific Function</a>. Schroeder C, Valenzuela J, Mejia Natividad I, et al. Thu, 07 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractABurstofGeneticInnovationinDrosophilaActinRelatedProteinsforTestisSpecificFunction">Abstract</button></h5><div id="abstractABurstofGeneticInnovationinDrosophilaActinRelatedProteinsforTestisSpecificFunction" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Many cytoskeletal proteins perform fundamental biological processes and are evolutionarily ancient. For example, the superfamily of actin-related proteins (Arps) specialized early in eukaryotic evolution for diverse cellular roles in the cytoplasm and the nucleus. Despite its strict conservation across eukaryotes, we find that the Arp superfamily has undergone dramatic lineage-specific diversification in <span style="font-style:italic;">Drosophila</span>. Our phylogenomic analyses reveal four independent Arp gene duplications that occurred in the common ancestor of the <span style="font-style:italic;">obscura</span> group of <span style="font-style:italic;">Drosophila</span> and have been mostly preserved in this lineage. All four <span style="font-style:italic;">obscura</span>-specific Arp paralogs are predominantly expressed in the male germline and have evolved under positive selection. We focus our analyses on the divergent <span style="font-style:italic;">Arp2D</span> paralog, which arose via a retroduplication event from <span style="font-style:italic;">Arp2</span>, a component of the Arp2/3 complex that polymerizes branched actin networks. Computational modeling analyses suggest that Arp2D can replace Arp2 in the Arp2/3 complex and bind actin monomers. Together with the signature of positive selection, our findings suggest that Arp2D may augment Arp2’s functions in the male germline. Indeed, we find that Arp2D is expressed during and following male meiosis, where it localizes to distinct locations such as actin cones—specialized cytoskeletal structures that separate bundled spermatids into individual mature sperm. We hypothesize that this unprecedented burst of genetic innovation in cytoskeletal proteins may have been driven by the evolution of sperm heteromorphism in the <span style="font-style:italic;">obscura</span> group of <span style="font-style:italic;">Drosophila</span>.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/773/5614438?rss=1>Plasmodium vivax Malaria Viewed through the Lens of an Eradicated European Strain</a>. van Dorp L, Gelabert P, Rieux A, et al. Thu, 07 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPlasmodiumvivaxMalariaViewedthroughtheLensofanEradicatedEuropeanStrain">Abstract</button></h5><div id="abstractPlasmodiumvivaxMalariaViewedthroughtheLensofanEradicatedEuropeanStrain" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>The protozoan <span style="font-style:italic;">Plasmodium vivax</span> is responsible for 42% of all cases of malaria outside Africa. The parasite is currently largely restricted to tropical and subtropical latitudes in Asia, Oceania, and the Americas. Though, it was historically present in most of Europe before being finally eradicated during the second half of the 20th century. The lack of genomic information on the extinct European lineage has prevented a clear understanding of historical population structuring and past migrations of <span style="font-style:italic;">P. vivax</span>. We used medical microscope slides prepared in 1944 from malaria-affected patients from the Ebro Delta in Spain, one of the last footholds of malaria in Europe, to generate a genome of a European <span style="font-style:italic;">P. vivax</span> strain. Population genetics and phylogenetic analyses placed this strain basal to a cluster including samples from the Americas. This genome allowed us to calibrate a genomic mutation rate for <span style="font-style:italic;">P. vivax</span>, and to estimate the mean age of the last common ancestor between European and American strains to the 15th century. This date points to an introduction of the parasite during the European colonization of the Americas. In addition, we found that some known variants for resistance to antimalarial drugs, including Chloroquine and Sulfadoxine, were already present in this European strain, predating their use. Our results shed light on the evolution of an important human pathogen and illustrate the value of antique medical collections as a resource for retrieving genomic information on pathogens from the past.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/695/5613842?rss=1>Divergent Selection and Primary Gene Flow Shape Incipient Speciation of a Riparian Tree on Hawaii Island</a>. Choi J, Purugganan M, Stacy E, et al. Wed, 06 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDivergentSelectionandPrimaryGeneFlowShapeIncipientSpeciationofaRiparianTreeonHawaiiIsland">Abstract</button></h5><div id="abstractDivergentSelectionandPrimaryGeneFlowShapeIncipientSpeciationofaRiparianTreeonHawaiiIsland" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>A long-standing goal of evolutionary biology is to understand the mechanisms underlying the formation of species. Of particular interest is whether or not speciation can occur in the presence of gene flow and without a period of physical isolation. Here, we investigated this process within Hawaiian <span style="font-style:italic;">Metrosideros</span>, a hypervariable and highly dispersible woody species complex that dominates the Hawaiian Islands in continuous stands. Specifically, we investigated the origin of <span style="font-style:italic;">Metrosideros polymorpha</span> var. <span style="font-style:italic;">newellii</span> (newellii), a riparian ecotype endemic to Hawaii Island that is purportedly derived from the archipelago-wide <span style="font-style:italic;">M. polymorpha</span> var. <span style="font-style:italic;">glaberrima</span> (glaberrima). Disruptive selection across a sharp forest-riparian ecotone contributes to the isolation of these varieties and is a likely driver of newellii’s origin. We examined genome-wide variation of 42 trees from Hawaii Island and older islands. Results revealed a split between glaberrima and newellii within the past 0.3–1.2 My. Admixture was extensive between lineages within Hawaii Island and between islands, but introgression from populations on older islands (i.e., secondary gene flow) did not appear to contribute to the emergence of newellii. In contrast, recurrent gene flow (i.e., primary gene flow) between glaberrima and newellii contributed to the formation of genomic islands of elevated absolute and relative divergence. These regions were enriched for genes with regulatory functions as well as for signals of positive selection, especially in newellii, consistent with divergent selection underlying their formation. In sum, our results support riparian newellii as a rare case of incipient ecological speciation with primary gene flow in trees.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/711/5613187?rss=1>Genomic Resources for Darters (Percidae: Etheostominae) Provide Insight into Postzygotic Barriers Implicated in Speciation</a>. Moran R, Catchen J, Fuller R, et al. Tue, 05 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGenomicResourcesforDartersPercidaeEtheostominaeProvideInsightintoPostzygoticBarriersImplicatedinSpeciation">Abstract</button></h5><div id="abstractGenomicResourcesforDartersPercidaeEtheostominaeProvideInsightintoPostzygoticBarriersImplicatedinSpeciation" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Comparative genomic approaches are increasingly being used to study the evolution of reproductive barriers in nonmodel species. Although numerous studies have examined prezygotic isolation in darters (Percidae), investigations into postzygotic barriers have remained rare due to long generation times and a lack of genomic resources. Orangethroat and rainbow darters naturally hybridize and provide a remarkable example of male-driven speciation via character displacement. Backcross hybrids suffer from high mortality, which appears to promote behavioral isolation in sympatry. To investigate the genomic architecture of postzygotic isolation, we used Illumina and PacBio sequencing to generate a chromosome-level, annotated assembly of the orangethroat darter genome and high-density linkage maps for orangethroat and rainbow darters. We also analyzed genome-wide RADseq data from wild-caught adults of both species and laboratory-generated backcrosses to identify genomic regions associated with hybrid incompatibles. Several putative chromosomal translocations and inversions were observed between orangethroat and rainbow darters, suggesting structural rearrangements may underlie postzygotic isolation. We also found evidence of selection against recombinant haplotypes and transmission ratio distortion in backcross hybrid genomes, providing further insight into the genomic architecture of genetic incompatibilities. Notably, regions with high levels of genetic divergence between species were enriched for genes associated with developmental and meiotic processes, providing strong candidates for postzygotic isolating barriers. These findings mark significant contributions to our understanding of the genetic basis of reproductive isolation between species undergoing character displacement. Furthermore, the genomic resources presented here will be instrumental for studying speciation in darters, the most diverse vertebrate group in North America.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/893/5607305?rss=1>A Bayesian Framework for Inferring the Influence of Sequence Context on Point Mutations</a>. Ling G, Miller D, Nielsen R, et al. Tue, 05 Nov 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractABayesianFrameworkforInferringtheInfluenceofSequenceContextonPointMutations">Abstract</button></h5><div id="abstractABayesianFrameworkforInferringtheInfluenceofSequenceContextonPointMutations" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>The probability of point mutations is expected to be highly influenced by the flanking nucleotides that surround them, known as the sequence context. This phenomenon may be mainly attributed to the enzyme that modifies or mutates the genetic material, because most enzymes tend to have specific sequence contexts that dictate their activity. Here, we develop a statistical model that allows for the detection and evaluation of the effects of different sequence contexts on mutation rates from deep population sequencing data. This task is computationally challenging, as the complexity of the model increases exponentially as the context size increases. We established our novel Bayesian method based on sparse model selection methods, with the leading assumption that the number of actual sequence contexts that directly influence mutation rates is minuscule compared with the number of possible sequence contexts. We show that our method is highly accurate on simulated data using pentanucleotide contexts, even when accounting for noisy data. We next analyze empirical population sequencing data from polioviruses and HIV-1 and detect a significant enrichment in sequence contexts associated with deamination by the cellular deaminases ADAR 1/2 and APOBEC3G, respectively. In the current era, where next-generation sequencing data are highly abundant, our approach can be used on any population sequencing data to reveal context-dependent base alterations and may assist in the discovery of novel mutable sites or editing sites.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/683/5610533?rss=1>Mitochondrial Metagenomics Reveals the Ancient Origin and Phylodiversity of Soil Mites and Provides a Phylogeny of the Acari</a>. Arribas P, Andújar C, Moraza M, et al. Thu, 31 Oct 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractMitochondrialMetagenomicsRevealstheAncientOriginandPhylodiversityofSoilMitesandProvidesaPhylogenyoftheAcari">Abstract</button></h5><div id="abstractMitochondrialMetagenomicsRevealstheAncientOriginandPhylodiversityofSoilMitesandProvidesaPhylogenyoftheAcari" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>High-throughput DNA methods hold great promise for phylogenetic analysis of lineages that are difficult to study with conventional molecular and morphological approaches. The mites (Acari), and in particular the highly diverse soil-dwelling lineages, are among the least known branches of the metazoan Tree-of-Life. We extracted numerous minute mites from soils in an area of mixed forest and grassland in southern Iberia. Selected specimens representing the full morphological diversity were shotgun sequenced in bulk, followed by genome assembly of short reads from the mixture, which produced &gt;100 mitochondrial genomes representing diverse acarine lineages. Phylogenetic analyses in combination with taxonomically limited mitogenomes available publicly resulted in plausible trees defining basal relationships of the Acari. Several critical nodes were supported by ancestral-state reconstructions of mitochondrial gene rearrangements. Molecular calibration placed the minimum age for the common ancestor of the superorder Acariformes, which includes most soil-dwelling mites, to the Cambrian–Ordovician (likely within 455–552 Ma), whereas the origin of the superorder Parasitiformes was placed later in the Carboniferous-Permian. Most family-level taxa within the Acariformes were dated to the Jurassic and Triassic. The ancient origin of Acariformes and the early diversification of major extant lineages linked to the soil are consistent with a pioneering role for mites in building the earliest terrestrial ecosystems.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/660/5607304?rss=1>OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States</a>. Anand A, Chen K, Catoiu E, et al. Fri, 25 Oct 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractOxyRIsaConvergentTargetforMutationsAcquiredduringAdaptationtoOxidativeStressProneMetabolicStates">Abstract</button></h5><div id="abstractOxyRIsaConvergentTargetforMutationsAcquiredduringAdaptationtoOxidativeStressProneMetabolicStates" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Oxidative stress is concomitant with aerobic metabolism. Thus, bacterial genomes encode elaborate mechanisms to achieve redox homeostasis. Here we report that the peroxide-sensing transcription factor, <span style="font-style:italic;">oxyR</span>, is a common mutational target using bacterial species belonging to two genera, <span style="font-style:italic;">Escherichia coli</span> and <span style="font-style:italic;">Vibrio natriegens</span>, in separate growth conditions implemented during laboratory evolution. The mutations clustered in the redox active site, dimer interface, and flexible redox loop of the protein. These mutations favor the oxidized conformation of OxyR that results in constitutive expression of the genes it regulates. Independent component analysis of the transcriptome revealed that the constitutive activity of OxyR reduces DNA damage from reactive oxygen species, as inferred from the activity of the SOS response regulator LexA. This adaptation to peroxide stress came at a cost of lower growth, as revealed by calculations of proteome allocation using genome-scale models of metabolism and macromolecular expression. Further, identification of similar sequence changes in natural isolates of <span style="font-style:italic;">E. coli</span> indicates that adaptation to oxidative stress through genetic changes in <span style="font-style:italic;">oxyR</span> can be a common occurrence.</span></div>
<h5><b>[MolBiolEvol]</b> <a href=https://academic.oup.com/mbe/article/37/3/668/5607302?rss=1>Higher Gene Flow in Sex-Related Chromosomes than in Autosomes during Fungal Divergence</a>. Hartmann F, Rodríguez de la Vega R, Gladieux P, et al. Fri, 25 Oct 2019  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractHigherGeneFlowinSexRelatedChromosomesthaninAutosomesduringFungalDivergence">Abstract</button></h5><div id="abstractHigherGeneFlowinSexRelatedChromosomesthaninAutosomesduringFungalDivergence" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div>Nonrecombining sex chromosomes are widely found to be more differentiated than autosomes among closely related species, due to smaller effective population size and/or to a disproportionally large-X effect in reproductive isolation. Although fungal mating-type chromosomes can also display large nonrecombining regions, their levels of differentiation compared with autosomes have been little studied. Anther-smut fungi from the <span style="font-style:italic;">Microbotryum</span> genus are castrating pathogens of Caryophyllaceae plants with largely nonrecombining mating-type chromosomes. Using whole genome sequences of 40 fungal strains, we quantified genetic differentiation among strains isolated from the geographically overlapping North American species and subspecies of <span style="font-style:italic;">Silene virginica</span> and <span style="font-style:italic;">S. caroliniana</span>. We inferred that gene flow likely occurred at the early stages of divergence and then completely stopped. We identified large autosomal genomic regions with chromosomal inversions, with higher genetic divergence than the rest of the genomes and highly enriched in selective sweeps, supporting a role of rearrangements in preventing gene flow in genomic regions involved in ecological divergence. Unexpectedly, the nonrecombining mating-type chromosomes showed lower divergence than autosomes due to higher gene flow, which may be promoted by adaptive introgressions of less degenerated mating-type chromosomes. The fact that both mating-type chromosomes are always heterozygous and nonrecombining may explain such patterns that oppose to those found for XY or ZW sex chromosomes. The specific features of mating-type chromosomes may also apply to the UV sex chromosomes determining sexes at the haploid stage in algae and bryophytes and may help test general hypotheses on the evolutionary specificities of sex-related chromosomes.</span></div><hr>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/155?rss=1>A HOTAIR regulatory element modulates glioma cell sensitivity to temozolomide through long-range regulation of multiple target genes [RESEARCH]</a>. Zhang, L., He, A., Chen, B., Bi, J., Chen, J., Guo.., Zhao, Z., Shi, J., An, W., Attenello, F., Lu, W. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAHOTAIRregulatoryelementmodulatesgliomacellsensitivitytotemozolomidethroughlongrangeregulationofmultipletargetgenesRESEARCH">Abstract</button></h5><div id="abstractAHOTAIRregulatoryelementmodulatesgliomacellsensitivitytotemozolomidethroughlongrangeregulationofmultipletargetgenesRESEARCH" class="collapse"><p>Temozolomide (TMZ) is a frequently used chemotherapy for glioma; however, chemoresistance is a major problem limiting its effectiveness. Thus, knowledge of mechanisms underlying this outcome could improve patient prognosis. Here, we report that deletion of a regulatory element in the <I>HOTAIR</I> locus increases glioma cell sensitivity to TMZ and alters transcription of multiple genes. Analysis of a combination of RNA-seq, Capture Hi-C, and patient survival data suggests that <I>CALCOCO1</I> and <I>ZC3H10</I> are target genes repressed by the <I>HOTAIR</I> regulatory element and that both function in regulating glioma cell sensitivity to TMZ. Rescue experiments and 3C data confirmed this hypothesis. We propose a new regulatory mechanism governing glioma cell TMZ sensitivity.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/164?rss=1>An efficient RNA-seq-based segregation analysis identifies the sex chromosomes of Cannabis sativa [RESEARCH]</a>. Prentout, D., Razumova, O., Rhone, B., Badouin, H...., Ka&#x0308;fer, J., Karlov, G., Marais, G. A. B. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAnefficientRNAseqbasedsegregationanalysisidentifiesthesexchromosomesofCannabissativaRESEARCH">Abstract</button></h5><div id="abstractAnefficientRNAseqbasedsegregationanalysisidentifiesthesexchromosomesofCannabissativaRESEARCH" class="collapse"><p><I>Cannabis</I> <I>sativa</I>&ndash;derived tetrahydrocannabinol (THC) production is increasing very fast worldwide. <I>C. sativa</I> is a dioecious plant with XY Chromosomes, and only females (XX) are useful for THC production. Identifying the sex chromosome sequence would improve early sexing and better management of this crop; however, the <I>C. sativa</I> genome projects have failed to do so. Moreover, as dioecy in the Cannabaceae family is ancestral, <I>C. sativa</I> sex chromosomes are potentially old and thus very interesting to study, as little is known about old plant sex chromosomes. Here, we RNA-sequenced a <I>C. sativa</I> family (two parents and 10 male and female offspring, 576 million reads) and performed a segregation analysis for all <I>C. sativa</I> genes using the probabilistic method SEX-DETector. We identified &gt;500 sex-linked genes. Mapping of these sex-linked genes to a <I>C. sativa</I> genome assembly identified the largest chromosome pair being the sex chromosomes. We found that the X-specific region (not recombining between X and Y) is large compared to other plant systems. Further analysis of the sex-linked genes revealed that <I>C. sativa</I> has a strongly degenerated Y Chromosome and may represent the oldest plant sex chromosome system documented so far. Our study revealed that old plant sex chromosomes can have large, highly divergent nonrecombining regions, yet still be roughly homomorphic.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/173?rss=1>Genome-wide redistribution of 24-nt siRNAs in rice gametes [RESEARCH]</a>. Li, C., Xu, H., Fu, F.-F., Russell, S. D., Sundaresan, V., Gent, J. I. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGenomewideredistributionof24ntsiRNAsinricegametesRESEARCH">Abstract</button></h5><div id="abstractGenomewideredistributionof24ntsiRNAsinricegametesRESEARCH" class="collapse"><p>Gametes constitute a critical stage of the plant life cycle during which the genome undergoes reprogramming in preparation for embryogenesis. Here, we examined genome-wide distributions of small RNAs in the sperm and egg cells of rice. We found that 24-nt siRNAs, which are a hallmark of RNA-directed DNA methylation (RdDM) in plants, were depleted from heterochromatin boundaries in both gametes relative to vegetative tissues, reminiscent of siRNA patterns in DDM1-type nucleosome remodeler mutants. In sperm cells, 24-nt siRNAs were spread across heterochromatic regions, while in egg cells, 24-nt siRNAs were concentrated at a smaller number of heterochromatic loci throughout the genome, especially at loci which also produced siRNAs in other tissues. In both gametes, patterns of CHH methylation, typically a strong indicator of RdDM, were similar to vegetative tissues, although lower in magnitude. These findings indicate that the small RNA transcriptome undergoes large-scale redistribution in both male and female gametes, which is not correlated with recruitment of DNA methyltransferases in gametes and suggestive of unexplored regulatory activities of gamete small RNAs.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/185?rss=1>Ancestry-agnostic estimation of DNA sample contamination from sequence reads [METHOD]</a>. Zhang, F., Flickinger, M., Taliun, S. A. G., InPSY..roll, S. A., Pato, C. N., Boehnke, M., Kang, H. M. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAncestryagnosticestimationofDNAsamplecontaminationfromsequencereadsMETHOD">Abstract</button></h5><div id="abstractAncestryagnosticestimationofDNAsamplecontaminationfromsequencereadsMETHOD" class="collapse"><p>Detecting and estimating DNA sample contamination are important steps to ensure high-quality genotype calls and reliable downstream analysis. Existing methods rely on population allele frequency information for accurate estimation of contamination rates. Correctly specifying population allele frequencies for each individual in early stage of sequence analysis is impractical or even impossible for large-scale sequencing centers that simultaneously process samples from multiple studies across diverse populations. On the other hand, incorrectly specified allele frequencies may result in substantial bias in estimated contamination rates. For example, we observed that existing methods often fail to identify 10% contaminated samples at a typical 3% contamination exclusion threshold when genetic ancestry is misspecified. Such an incomplete screening of contaminated samples substantially inflates the estimated rate of genotyping errors even in deeply sequenced genomes and exomes. We propose a robust statistical method that accurately estimates DNA contamination and is agnostic to genetic ancestry of the intended or contaminating sample. Our method integrates the estimation of genetic ancestry and DNA contamination in a unified likelihood framework by leveraging individual-specific allele frequencies projected from reference genotypes onto principal component coordinates. Our method can also be used for estimating genetic ancestries, similar to LASER or <I>TRACE</I>, but simultaneously accounting for potential contamination. We demonstrate that our method robustly estimates contamination rates and genetic ancestries across populations and contamination scenarios. We further demonstrate that, in the presence of contamination, genetic ancestry inference can be substantially biased with existing methods that ignore contamination, while our method corrects for such biases.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/195?rss=1>netNMF-sc: leveraging gene-gene interactions for imputation and dimensionality reduction in single-cell expression analysis [METHOD]</a>. Elyanow, R., Dumitrascu, B., Engelhardt, B. E., Raphael, B. J. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractnetNMFscleveraginggenegeneinteractionsforimputationanddimensionalityreductioninsinglecellexpressionanalysisMETHOD">Abstract</button></h5><div id="abstractnetNMFscleveraginggenegeneinteractionsforimputationanddimensionalityreductioninsinglecellexpressionanalysisMETHOD" class="collapse"><p>Single-cell RNA-sequencing (scRNA-seq) enables high-throughput measurement of RNA expression in single cells. However, because of technical limitations, scRNA-seq data often contain zero counts for many transcripts in individual cells. These zero counts, or dropout events, complicate the analysis of scRNA-seq data using standard methods developed for bulk RNA-seq data. Current scRNA-seq analysis methods typically overcome dropout by combining information across cells in a lower-dimensional space, leveraging the observation that cells generally occupy a small number of RNA expression states. We introduce netNMF-sc, an algorithm for scRNA-seq analysis that leverages information across both cells and genes. netNMF-sc learns a low-dimensional representation of scRNA-seq transcript counts using network-regularized non-negative matrix factorization. The network regularization takes advantage of prior knowledge of gene&ndash;gene interactions, encouraging pairs of genes with known interactions to be nearby each other in the low-dimensional representation. The resulting matrix factorization imputes gene abundance for both zero and nonzero counts and can be used to cluster cells into meaningful subpopulations. We show that netNMF-sc outperforms existing methods at clustering cells and estimating gene&ndash;gene covariance using both simulated and real scRNA-seq data, with increasing advantages at higher dropout rates (e.g., &gt;60%). We also show that the results from netNMF-sc are robust to variation in the input network, with more representative networks leading to greater performance gains.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/205?rss=1>SHARP: hyperfast and accurate processing of single-cell RNA-seq data via ensemble random projection [METHOD]</a>. Wan, S., Kim, J., Won, K. J. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractSHARPhyperfastandaccurateprocessingofsinglecellRNAseqdataviaensemblerandomprojectionMETHOD">Abstract</button></h5><div id="abstractSHARPhyperfastandaccurateprocessingofsinglecellRNAseqdataviaensemblerandomprojectionMETHOD" class="collapse"><p>To process large-scale single-cell RNA-sequencing (scRNA-seq) data effectively without excessive distortion during dimension reduction, we present SHARP, an ensemble random projection-based algorithm that is scalable to clustering 10 million cells. Comprehensive benchmarking tests on 17 public scRNA-seq data sets show that SHARP outperforms existing methods in terms of speed and accuracy. Particularly, for large-size data sets (more than 40,000 cells), SHARP runs faster than other competitors while maintaining high clustering accuracy and robustness. To the best of our knowledge, SHARP is the only R-based tool that is scalable to clustering scRNA-seq data with 10 million cells.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/214?rss=1>Deep neural networks for interpreting RNA-binding protein target preferences [METHOD]</a>. Ghanbari, M., Ohler, U. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDeepneuralnetworksforinterpretingRNAbindingproteintargetpreferencesMETHOD">Abstract</button></h5><div id="abstractDeepneuralnetworksforinterpretingRNAbindingproteintargetpreferencesMETHOD" class="collapse"><p>Deep learning has become a powerful paradigm to analyze the binding sites of regulatory factors including RNA-binding proteins (RBPs), owing to its strength to learn complex features from possibly multiple sources of raw data. However, the interpretability of these models, which is crucial to improve our understanding of RBP binding preferences and functions, has not yet been investigated in significant detail. We have designed a multitask and multimodal deep neural network for characterizing in vivo RBP targets. The model incorporates not only the sequence but also the region type of the binding sites as input, which helps the model to boost the prediction performance. To interpret the model, we quantified the contribution of the input features to the predictive score of each RBP. Learning across multiple RBPs at once, we are able to avoid experimental biases and to identify the RNA sequence motifs and transcript context patterns that are the most important for the predictions of each individual RBP. Our findings are consistent with known motifs and binding behaviors and can provide new insights about the regulatory functions of RBPs.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/227?rss=1>MOCHI enables discovery of heterogeneous interactome modules in 3D nucleome [METHOD]</a>. Tian, D., Zhang, R., Zhang, Y., Zhu, X., Ma, J. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractMOCHIenablesdiscoveryofheterogeneousinteractomemodulesin3DnucleomeMETHOD">Abstract</button></h5><div id="abstractMOCHIenablesdiscoveryofheterogeneousinteractomemodulesin3DnucleomeMETHOD" class="collapse"><p>The composition of the cell nucleus is highly heterogeneous, with different constituents forming complex interactomes. However, the global patterns of these interwoven heterogeneous interactomes remain poorly understood. Here we focus on two different interactomes, chromatin interaction network and gene regulatory network, as a proof of principle to identify heterogeneous interactome modules (HIMs), each of which represents a cluster of gene loci that is in spatial contact more frequently than expected and that is regulated by the same group of transcription factors. HIM integrates transcription factor binding and 3D genome structure to reflect "transcriptional niche" in the nucleus. We develop a new algorithm, MOCHI, to facilitate the discovery of HIMs based on network motif clustering in heterogeneous interactomes. By applying MOCHI to five different cell types, we found that HIMs have strong spatial preference within the nucleus and show distinct functional properties. Through integrative analysis, this work shows the utility of MOCHI to identify HIMs, which may provide new perspectives on the interplay between transcriptional regulation and 3D genome organization.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/239?rss=1>TraDIS-Xpress: a high-resolution whole-genome assay identifies novel mechanisms of triclosan action and resistance [METHOD]</a>. Yasir, M., Turner, A. K., Bastkowski, S., Baker, D.. G. M., Darling, A., Webber, M. A., Charles, I. G. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTraDISXpressahighresolutionwholegenomeassayidentifiesnovelmechanismsoftriclosanactionandresistanceMETHOD">Abstract</button></h5><div id="abstractTraDISXpressahighresolutionwholegenomeassayidentifiesnovelmechanismsoftriclosanactionandresistanceMETHOD" class="collapse"><p>Understanding the genetic basis for a phenotype is a central goal in biological research. Much has been learnt about bacterial genomes by creating large mutant libraries and looking for conditionally important genes. However, current genome-wide methods are largely unable to assay essential genes which are not amenable to disruption. To overcome this limitation, we developed a new version of "TraDIS" (transposon directed insertion-site sequencing) that we term "TraDIS-Xpress" that combines an inducible promoter into the transposon cassette. This allows controlled overexpression and repression of all genes owing to saturation of inserts adjacent to all open reading frames as well as conventional inactivation. We applied TraDIS-Xpress to identify responses to the biocide triclosan across a range of concentrations. Triclosan is endemic in modern life, but there is uncertainty about its mode of action with a concentration-dependent switch from bacteriostatic to bactericidal action unexplained. Our results show a concentration-dependent response to triclosan with different genes important in survival between static and cidal exposures. These genes include those previously reported to have a role in triclosan resistance as well as a new set of genes, including essential genes. Novel genes identified as being sensitive to triclosan exposure include those involved in barrier function, small molecule uptake, and integrity of transcription and translation. We anticipate the approach we show here, by allowing comparisons across multiple experimental conditions of TraDIS data, and including essential genes, will be a starting point for future work examining how different drug conditions impact bacterial survival mechanisms.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/250?rss=1>A comparison of gene expression and DNA methylation patterns across tissues and species [RESOURCES]</a>. Blake, L. E., Roux, J., Hernando-Herraez, I., Bano..Eres, I., Cuevas, C., Marques-Bonet, T., Gilad, Y. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAcomparisonofgeneexpressionandDNAmethylationpatternsacrosstissuesandspeciesRESOURCES">Abstract</button></h5><div id="abstractAcomparisonofgeneexpressionandDNAmethylationpatternsacrosstissuesandspeciesRESOURCES" class="collapse"><p>Previously published comparative functional genomic data sets from primates using frozen tissue samples, including many data sets from our own group, were often collected and analyzed using nonoptimal study designs and analysis approaches. In addition, when samples from multiple tissues were studied in a comparative framework, individuals and tissues were confounded. We designed a multitissue comparative study of gene expression and DNA methylation in primates that minimizes confounding effects by using a balanced design with respect to species, tissues, and individuals. We also developed a comparative analysis pipeline that minimizes biases attributable to sequence divergence. Thus, we present the most comprehensive catalog of similarities and differences in gene expression and DNA methylation levels between livers, kidneys, hearts, and lungs, in humans, chimpanzees, and rhesus macaques. We estimate that overall, interspecies and inter-tissue differences in gene expression levels can only modestly be accounted for by corresponding differences in promoter DNA methylation. However, the expression pattern of genes with conserved inter-tissue expression differences can be explained by corresponding interspecies methylation changes more often. Finally, we show that genes whose tissue-specific regulatory patterns are consistent with the action of natural selection are highly connected in both gene regulatory and protein&ndash;protein interaction networks.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/263?rss=1>A time-resolved multi-omic atlas of the developing mouse liver [RESOURCES]</a>. Gong, T., Zhang, C., Ni, X., Li, X., Li, J., Liu, ..., Song, L., Zhou, Q., Ding, C., Qin, J., Wang, Y. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAtimeresolvedmultiomicatlasofthedevelopingmouseliverRESOURCES">Abstract</button></h5><div id="abstractAtimeresolvedmultiomicatlasofthedevelopingmouseliverRESOURCES" class="collapse"><p>Liver organogenesis and development are composed of a series of complex, well-orchestrated events. Identifying key factors and pathways governing liver development will help elucidate the physiological and pathological processes including those of cancer. We conducted multidimensional omics measurements including protein, mRNA, and transcription factor (TF) DNA-binding activity for mouse liver tissues collected from embryonic day 12.5 (E12.5) to postnatal week 8 (W8), encompassing major developmental stages. These data sets reveal dynamic changes of core liver functions and canonical signaling pathways governing development at both mRNA and protein levels. The TF DNA-binding activity data set highlights the importance of TF activity in early embryonic development. A comparison between mouse liver development and human hepatocellular carcinoma (HCC) proteomic profiles reveal that more aggressive tumors are characterized with the activation of early embryonic development pathways, whereas less aggressive ones maintain liver function&ndash;related pathways that are elevated in the mature liver. This work offers a panoramic view of mouse liver development and provides a rich resource to explore in-depth functional characterization.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/276?rss=1>Organ-level protein networks as a reference for the host effects of the microbiome [RESOURCES]</a>. Mills, R. H., Wozniak, J. M., Vrbanac, A., Campeau..aing, B., Gewirtz, A., Knight, R., Gonzalez, D. J. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractOrganlevelproteinnetworksasareferenceforthehosteffectsofthemicrobiomeRESOURCES">Abstract</button></h5><div id="abstractOrganlevelproteinnetworksasareferenceforthehosteffectsofthemicrobiomeRESOURCES" class="collapse"><p>Connections between the microbiome and health are rapidly emerging in a wide range of diseases. However, a detailed mechanistic understanding of how different microbial communities are influencing their hosts is often lacking. One method researchers have used to understand these effects are germ-free (GF) mouse models. Differences found within the organ systems of these model organisms may highlight generalizable mechanisms that microbiome dysbioses have throughout the host. Here, we applied multiplexed, quantitative proteomics on the brains, spleens, hearts, small intestines, and colons of conventionally raised and GF mice, identifying associations to colonization state in over 7000 proteins. Highly ranked associations were constructed into protein&ndash;protein interaction networks and visualized onto an interactive 3D mouse model for user-guided exploration. These results act as a resource for microbiome researchers hoping to identify host effects of microbiome colonization on a given organ of interest. Our results include validation of previously reported effects in xenobiotic metabolism, the innate immune system, and glutamate-associated proteins while simultaneously providing organism-wide context. We highlight organism-wide differences in mitochondrial proteins including consistent increases in NNT, a mitochondrial protein with essential roles in influencing levels of NADH and NADPH, in all analyzed organs of conventional mice. Our networks also reveal new associations for further exploration, including protease responses in the spleen, high-density lipoproteins in the heart, and glutamatergic signaling in the brain. In total, our study provides a resource for microbiome researchers through detailed tables and visualization of the protein-level effects of microbial colonization on several organ systems.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/287?rss=1>Direct full-length RNA sequencing reveals unexpected transcriptome complexity during Caenorhabditis elegans development [RESOURCES]</a>. Li, R., Ren, X., Ding, Q., Bi, Y., Xie, D., Zhao, Z. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDirectfulllengthRNAsequencingrevealsunexpectedtranscriptomecomplexityduringCaenorhabditiselegansdevelopmentRESOURCES">Abstract</button></h5><div id="abstractDirectfulllengthRNAsequencingrevealsunexpectedtranscriptomecomplexityduringCaenorhabditiselegansdevelopmentRESOURCES" class="collapse"><p>Massively parallel sequencing of the polyadenylated RNAs has played a key role in delineating transcriptome complexity, including alternative use of an exon, promoter, 5' or 3' splice site or polyadenylation site, and RNA modification. However, reads derived from the current RNA-seq technologies are usually short and deprived of information on modification, compromising their potential in defining transcriptome complexity. Here, we applied a direct RNA sequencing method with ultralong reads using Oxford Nanopore Technologies to study the transcriptome complexity in <I>Caenorhabditis elegans</I>. We generated approximately six million reads using native poly(A)-tailed mRNAs from three developmental stages, with average read lengths ranging from 900 to 1100 nt. Around half of the reads represent full-length transcripts. To utilize the full-length transcripts in defining transcriptome complexity, we devised a method to classify the long reads as the same as existing transcripts or as a novel transcript using sequence mapping tracks rather than existing intron/exon structures, which allowed us to identify roughly 57,000 novel isoforms and recover at least 26,000 out of the 33,500 existing isoforms. The sets of genes with differential expression versus differential isoform usage over development are largely different, implying a fine-tuned regulation at isoform level. We also observed an unexpected increase in putative RNA modification in all bases in the coding region relative to the UTR, suggesting their possible roles in translation. The RNA reads and the method for read classification are expected to deliver new insights into RNA processing and modification and their underlying biology in the future.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/299?rss=1>The full-length transcriptome of C. elegans using direct RNA sequencing [RESOURCES]</a>. Roach, N. P., Sadowski, N., Alessi, A. F., Timp, W., Taylor, J., Kim, J. K. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractThefulllengthtranscriptomeofCelegansusingdirectRNAsequencingRESOURCES">Abstract</button></h5><div id="abstractThefulllengthtranscriptomeofCelegansusingdirectRNAsequencingRESOURCES" class="collapse"><p>Current transcriptome annotations have largely relied on short read lengths intrinsic to the most widely used high-throughput cDNA sequencing technologies. For example, in the annotation of the <I>Caenorhabditis elegans</I> transcriptome, more than half of the transcript isoforms lack full-length support and instead rely on inference from short reads that do not span the full length of the isoform. We applied nanopore-based direct RNA sequencing to characterize the developmental polyadenylated transcriptome of <I>C. elegans</I>. Taking advantage of long reads spanning the full length of mRNA transcripts, we provide support for 23,865 splice isoforms across 14,611 genes, without the need for computational reconstruction of gene models. Of the isoforms identified, 3452 are novel splice isoforms not present in the WormBase WS265 annotation. Furthermore, we identified 16,342 isoforms in the 3' untranslated region (3' UTR), 2640 of which are novel and do not fall within 10 bp of existing 3'-UTR data sets and annotations. Combining 3' UTRs and splice isoforms, we identified 28,858 full-length transcript isoforms. We also determined that poly(A) tail lengths of transcripts vary across development, as do the strengths of previously reported correlations between poly(A) tail length and expression level, and poly(A) tail length and 3'-UTR length. Finally, we have formatted this data as a publicly accessible track hub, enabling researchers to explore this data set easily in a genome browser.</p></div>
<h5><b>[GenomeRes]</b> <a href=http://genome.cshlp.org/cgi/content/short/30/2/313?rss=1>Corrigendum: Retrotransposition creates sloping shores: a graded influence of hypomethylated CpG islands on flanking CpG sites [CORRIGENDUM]</a>. Grandi, F. C., Rosser, J. M., Newkirk, S. J., Yin,.., Ivics, Z., Izsvak, Z., Ye, P., Yu, Y. E., An, W. 2020-02-20T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCorrigendumRetrotranspositioncreatesslopingshoresagradedinfluenceofhypomethylatedCpGislandsonflankingCpGsitesCORRIGENDUM">Abstract</button></h5><div id="abstractCorrigendumRetrotranspositioncreatesslopingshoresagradedinfluenceofhypomethylatedCpGislandsonflankingCpGsitesCORRIGENDUM" class="collapse"></div><hr>
<h5><b>[GenomeBiol]</b> <a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01988-3>Benchmarking of computational error-correction methods for next-generation sequencing data</a>. Keith Mitchell, Jaqueline J. Brito, Igor Mandric, ..Yang, Kevin Hsieh, Linus Chen, Eli Littman&hellip; 2020-03-17T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractBenchmarkingofcomputationalerrorcorrectionmethodsfornextgenerationsequencingdata">Abstract</button></h5><div id="abstractBenchmarkingofcomputationalerrorcorrectionmethodsfornextgenerationsequencingdata" class="collapse">Recent advancements in next-generation sequencing have rapidly improved our ability to study genomic material at an unprecedented scale. Despite substantial improvements in sequencing technologies, errors pres...</div>
<h5><b>[GenomeBiol]</b> <a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01967-8>BANDITS: Bayesian differential splicing accounting for sample-to-sample variability and mapping uncertainty</a>. Simone Tiberi and Mark D. Robinson 2020-03-16T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractBANDITSBayesiandifferentialsplicingaccountingforsampletosamplevariabilityandmappinguncertainty">Abstract</button></h5><div id="abstractBANDITSBayesiandifferentialsplicingaccountingforsampletosamplevariabilityandmappinguncertainty" class="collapse">Alternative splicing is a biological process during gene expression that allows a single gene to code for multiple proteins. However, splicing patterns can be altered in some conditions or diseases. Here, we p...</div>
<h5><b>[GenomeBiol]</b> <a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01974-9>Ultraplexing: increasing the efficiency of long-read sequencing for hybrid assembly with k-mer-based multiplexing</a>. Alexander T. Dilthey, Sebastian A. Meyer and Achim J. Kaasch 2020-03-14T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractUltraplexingincreasingtheefficiencyoflongreadsequencingforhybridassemblywithkmerbasedmultiplexing">Abstract</button></h5><div id="abstractUltraplexingincreasingtheefficiencyoflongreadsequencingforhybridassemblywithkmerbasedmultiplexing" class="collapse">Hybrid genome assembly has emerged as an important technique in bacterial genomics, but cost and labor requirements limit large-scale application. We present Ultraplexing, a method to improve per-sample sequen...</div>
<h5><b>[GenomeBiol]</b> <a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01985-6>Renaissance minds in 21st century science</a>. Itai Yanai and Martin Lercher 2020-03-13T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractRenaissancemindsin21stcenturyscience">Abstract</button></h5><div id="abstractRenaissancemindsin21stcenturyscience" class="collapse"></div>
<h5><b>[GenomeBiol]</b> <a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01963-y>CHOP: haplotype-aware path indexing in population graphs</a>. Tom Mokveld, Jasper Linthorst, Zaid Al-Ars, Henne Holstege and Marcel Reinders 2020-03-11T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCHOPhaplotypeawarepathindexinginpopulationgraphs">Abstract</button></h5><div id="abstractCHOPhaplotypeawarepathindexinginpopulationgraphs" class="collapse">The practical use of graph-based reference genomes depends on the ability to align reads to them. Performing substring queries to paths through these graphs lies at the core of this task. The combination of in...</div><hr>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/561?rss=1>CNVmap: A Method and Software To Detect and Map Copy Number Variants from Segregation Data [Methods, Technology, [amp   ] Resources]</a>. Falque, M., Jebreen, K., Paux, E., Knaak, C., Mezmouk, S., Martin, O. C. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCNVmapAMethodandSoftwareToDetectandMapCopyNumberVariantsfromSegregationDataMethodsTechnologyampResources">Abstract</button></h5><div id="abstractCNVmapAMethodandSoftwareToDetectandMapCopyNumberVariantsfromSegregationDataMethodsTechnologyampResources" class="collapse"><p>Single nucleotide polymorphisms (SNPs) are used widely for detecting quantitative trait loci, or for searching for causal variants of diseases. Nevertheless, <I>structural</I> variations such as copy-number variants (CNVs) represent a large part of natural genetic diversity, and contribute significantly to trait variation. Numerous methods and softwares based on different technologies (amplicons, CGH, tiling, or SNP arrays, or sequencing) have already been developed to detect CNVs, but they bypass a wealth of information such as genotyping data from segregating populations, produced, <I>e.g.</I>, for QTL mapping. Here, we propose an original method to both detect and <I>genetically map</I> CNVs using mapping panels. Specifically, we exploit the apparent heterozygous state of duplicated loci: peaks in appropriately defined genome-wide allelic profiles provide highly specific signatures that identify the nature and position of the CNVs. Our original method and software can detect and map automatically up to 33 different predefined types of CNVs based on segregation data only. We validate this approach on simulated and experimental biparental mapping panels in two maize populations and one wheat population. Most of the events found correspond to having just one extra copy in one of the parental lines, but the corresponding allelic value can be that of either parent. We also find cases with two or more additional copies, especially in wheat, where these copies locate to homeologues. More generally, our computational tool can be used to give additional value, at no cost, to many datasets produced over the past decade from genetic mapping panels.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/589?rss=1>Promotion of Hyperthermic-Induced rDNA Hypercondensation in Saccharomyces cerevisiae [Genome Integrity and Transmission]</a>. Shen, D., Skibbens, R. V. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPromotionofHyperthermicInducedrDNAHypercondensationinSaccharomycescerevisiaeGenomeIntegrityandTransmission">Abstract</button></h5><div id="abstractPromotionofHyperthermicInducedrDNAHypercondensationinSaccharomycescerevisiaeGenomeIntegrityandTransmission" class="collapse"><p>Ribosome biogenesis is tightly regulated through stress-sensing pathways that impact genome stability, aging and senescence. In <I>Saccharomyces cerevisiae</I>, ribosomal RNAs are transcribed from rDNA located on the right arm of chromosome&nbsp;XII. Numerous studies reveal that rDNA decondenses into a puff-like structure during interphase, and condenses into a tight loop-like structure during mitosis. Intriguingly, a novel and additional mechanism of increased mitotic rDNA compaction (termed hypercondensation) was recently discovered that occurs in response to temperature stress (hyperthermic-induced) and is rapidly reversible. Here, we report that neither changes in condensin binding or release of DNA during mitosis, nor mutation of factors that regulate cohesin binding and release, appear to play a critical role in hyperthermic-induced rDNA hypercondensation. A candidate genetic approach revealed that deletion of either <A HREF="https://identifiers.org/bioentitylink/SGD:S000006161?doi=10.1534/genetics.119.302994"><I>HSP82</I></inter-ref> or <inter-ref locator="https://identifiers.org/bioentitylink/SGD:S000004798?doi=10.1534/genetics.119.302994" locator-type="url"><I>HSC82</I></inter-ref> (Hsp90 encoding heat shock paralogs) result in significantly reduced hyperthermic-induced rDNA hypercondensation. Intriguingly, Hsp inhibitors do not impact rDNA hypercondensation. In combination, these findings suggest that Hsp90 either stabilizes client proteins, which are sensitive to very transient thermic challenges, or directly promotes rDNA hypercondensation during preanaphase. Our findings further reveal that the high mobility group protein <inter-ref locator="https://identifiers.org/bioentitylink/SGD:S000002581?doi=10.1534/genetics.119.302994" locator-type="url">Hmo1</A> is a negative regulator of mitotic rDNA condensation, distinct from its role in promoting premature condensation of rDNA during interphase upon nutrient starvation.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/651?rss=1>Impact of Chromosome Fusions on 3D Genome Organization and Gene Expression in Budding Yeast [Cellular Genetics]</a>. Di Stefano, M., Di Giovanni, F., Pozharskaia, V., .. D., Carey, L. B., Marti-Renom, M. A., Mendoza, M. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractImpactofChromosomeFusionson3DGenomeOrganizationandGeneExpressioninBuddingYeastCellularGenetics">Abstract</button></h5><div id="abstractImpactofChromosomeFusionson3DGenomeOrganizationandGeneExpressioninBuddingYeastCellularGenetics" class="collapse"><p>The three-dimensional (3D) organization of chromosomes can influence transcription. However, the frequency and magnitude of these effects remain debated. To determine how changes in chromosome positioning affect transcription across thousands of genes with minimal perturbation, we characterized nuclear organization and global gene expression in budding yeast containing chromosome fusions. We used computational modeling and single-cell imaging to determine chromosome positions, and integrated these data with genome-wide transcriptional profiles from RNA sequencing. We find that chromosome fusions dramatically alter 3D nuclear organization without leading to strong genome-wide changes in transcription. However, we observe a mild but significant and reproducible increase in the expression of genes displaced away from the periphery. The increase in transcription is inversely proportional to the propensity of a given locus to be at the nuclear periphery; for example, a 10% decrease in the propensity of a gene to reside at the nuclear envelope is accompanied by a 10% increase in gene expression. Modeling suggests that this is due to both deletion of telomeres and to displacement of genes relative to the nuclear periphery. These data suggest that basal transcriptional activity is sensitive to radial changes in gene position, and provide insight into the functional relevance of budding yeast chromosome-level 3D organization in gene expression.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/669?rss=1>New Roles for the Heterochronic Transcription Factor LIN-29 in Cuticle Maintenance and Lipid Metabolism at the Larval-to-Adult Transition in Caenorhabditis elegans [Developmental and Behavioral Genetics]</a>. Abete-Luzi, P., Fukushige, T., Yun, S., Krause, M. W., Eisenmann, D. M. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractNewRolesfortheHeterochronicTranscriptionFactorLIN29inCuticleMaintenanceandLipidMetabolismattheLarvaltoAdultTransitioninCaenorhabditiselegansDevelopmentalandBehavioralGenetics">Abstract</button></h5><div id="abstractNewRolesfortheHeterochronicTranscriptionFactorLIN29inCuticleMaintenanceandLipidMetabolismattheLarvaltoAdultTransitioninCaenorhabditiselegansDevelopmentalandBehavioralGenetics" class="collapse"><p>Temporal regulation of gene expression is a crucial aspect of metazoan development. In the roundworm <I>Caenorhabditis elegans</I>, the heterochronic pathway controls multiple developmental events in a time-specific manner. The most downstream effector of this pathway, the zinc-finger transcription factor <A HREF="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860">LIN-29</inter-ref>, acts in the last larval stage (L4) to regulate elements of the larval-to-adult switch. Here, we explore new <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref> targets and their implications for this developmental transition. We used RNA-sequencing to identify genes differentially expressed between animals misexpressing <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref> at an early time point and control animals. Among 230 <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref>-activated genes, we found that genes encoding cuticle collagens were overrepresented. Interestingly, expression of <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url"><I>lin-29</I></inter-ref> and some of these collagens was increased in adults with cuticle damage, suggesting a previously unknown function for <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref> in adult cuticle maintenance. On the other hand, genes involved in fat metabolism were enriched among 350 <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref>-downregulated targets. We used mass spectrometry to assay lipid content in animals overexpressing <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref> and observed reduced fatty acid levels. Many <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref>-repressed genes are normally expressed in the intestine, suggesting cell-nonautonomous regulation. We identified several <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</inter-ref> upregulated genes encoding signaling molecules that may act as mediators in the regulation of intestinally expressed genes encoding fat metabolic enzymes and vitellogenins. Overall, our results support the model of <inter-ref locator="https://identifiers.org/bioentitylink/WB:WBGene00003015?doi=10.1534/genetics.119.302860" locator-type="url">LIN-29</A> as a major regulator of adult cuticle synthesis and integrity, and as the trigger for metabolic changes that take place at the important transition from rapid growth during larval life to slower growth and offspring production during adulthood.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/691?rss=1>A New Polygenic Model for Nonfamilial Colorectal Cancer Inheritance Based on the Genetic Architecture of the Azoxymethane-Induced Mouse Model [Genetics of Complex Traits]</a>. Bissahoyo, A. C., Xie, Y., Yang, L., Pearsall, R. ..el de Villena, F., Angel, J. M., Threadgill, D. W. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractANewPolygenicModelforNonfamilialColorectalCancerInheritanceBasedontheGeneticArchitectureoftheAzoxymethaneInducedMouseModelGeneticsofComplexTraits">Abstract</button></h5><div id="abstractANewPolygenicModelforNonfamilialColorectalCancerInheritanceBasedontheGeneticArchitectureoftheAzoxymethaneInducedMouseModelGeneticsofComplexTraits" class="collapse"><p>The azoxymethane model of colorectal cancer (CRC) was used to gain insights into the genetic heterogeneity of nonfamilial CRC. We observed significant differences in susceptibility parameters across 40 mouse inbred strains, with 6 new and 18 of 24 previously identified mouse CRC modifier alleles detected using genome-wide association analysis. Tumor incidence varied in F1 as well as intercrosses and backcrosses between resistant and susceptible strains. Analysis of inheritance patterns indicates that resistance to CRC development is inherited as a dominant characteristic genome-wide, and that susceptibility appears to occur in individuals lacking a large-effect, or sufficient numbers of small-effect, polygenic resistance alleles. Our results suggest a new polygenic model for inheritance of nonfamilial CRC, and that genetic studies in humans aimed at identifying individuals with elevated susceptibility should be pursued through the lens of absence of dominant resistance alleles rather than for the presence of susceptibility alleles.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/703?rss=1>The Pheromone and Pheromone Receptor Mating-Type Locus Is Involved in Controlling Uniparental Mitochondrial Inheritance in Cryptococcus [Genetics of Complex Traits]</a>. Sun, S., Fu, C., Ianiri, G., Heitman, J. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractThePheromoneandPheromoneReceptorMatingTypeLocusIsInvolvedinControllingUniparentalMitochondrialInheritanceinCryptococcusGeneticsofComplexTraits">Abstract</button></h5><div id="abstractThePheromoneandPheromoneReceptorMatingTypeLocusIsInvolvedinControllingUniparentalMitochondrialInheritanceinCryptococcusGeneticsofComplexTraits" class="collapse"><p>Mitochondria are inherited uniparentally during sexual reproduction in the majority of eukaryotic species studied, including humans, mice, and nematodes, as well as many fungal species. Mitochondrial uniparental inheritance (mito-UPI) could be beneficial in that it avoids possible genetic conflicts between organelles with different genetic backgrounds, as recently shown in mice, and it could prevent the spread of selfish genetic elements in the mitochondrial genome. Despite the prevalence of observed mito-UPI, the underlying mechanisms and the genes involved in controlling this non-Mendelian inheritance are poorly understood in many species. In <I>Cryptococcus neoformans</I>, a human pathogenic basidiomyceteous fungus, mating types (<I>MAT</I>&alpha; and <I>MAT</I><b>a</b>) are defined by alternate alleles at the single <I>MAT</I> locus that evolved from fusion of the two <I>MAT</I> loci (<I>P/R</I> encoding pheromones and pheromone receptors, and <I>HD</I> encoding homeodomain transcription factors) that are the ancestral state in the basidiomycota. Mitochondria are inherited uniparentally from the <I>MAT</I><b>a</b> parent in <I>C. neoformans</I>, and this requires the <I>SXI1</I>&alpha; and <I>SXI2</I><b>a</b> HD factors encoded by <I>MAT</I>. However, there is evidence that additional genes contribute to the control of mito-UPI in <I>Cryptococcus</I>. Here, we show that in <I>C. amylolentus</I>, a sibling species of <I>C. neoformans</I> with unlinked <I>P/R</I> and <I>HD MAT</I> loci, mito-UPI is controlled by the <I>P/R</I> locus and is independent of the <I>HD</I> locus. Consistently, by replacing the <I>MAT</I>&alpha; alleles of the pheromones (<I>MF</I>) and pheromone receptor (<I>STE3</I>) with the <I>MAT</I><b>a</b> alleles, we show that these <I>P/R</I> locus-defining genes indeed affect mito-UPI in <I>C. neoformans</I> during sexual reproduction. Additionally, we show that during early stages of <I>C. neoformans</I> sexual reproduction, conjugation tubes are always produced by the <I>MAT</I>&alpha; cells, resulting in unidirectional migration of the <I>MAT</I>&alpha; nucleus into the <I>MAT</I><b>a</b> cell during zygote formation. This process is controlled by the <I>P/R</I> locus and could serve to physically restrict movement of <I>MAT</I>&alpha; mitochondria in the zygotes, and thereby contribute to mito-UPI. We propose a model in which both physical and genetic mechanisms function in concert to prevent the coexistence of mitochondria from the two parents in the zygote, and subsequently in the meiotic progeny, thus ensuring mito-UPI in pathogenic <I>Cryptococcus</I>, as well as in closely related nonpathogenic species. The implications of these findings are discussed in the context of the evolution of mito-UPI in fungi and other more diverse eukaryotes.</p></div>
<h5><b>[Genetics]</b> <a href=http://www.genetics.org/cgi/content/short/214/3/719?rss=1>A Microbe Associated with Sleep Revealed by a Novel Systems Genetic Analysis of the Microbiome in Collaborative Cross Mice [Genetics of Complex Traits]</a>. Bubier, J. A., Philip, V. M., Quince, C., Campbell.., B. F., Palumbo, A. V., Podar, M., Chesler, E. J. 2020-03-04T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAMicrobeAssociatedwithSleepRevealedbyaNovelSystemsGeneticAnalysisoftheMicrobiomeinCollaborativeCrossMiceGeneticsofComplexTraits">Abstract</button></h5><div id="abstractAMicrobeAssociatedwithSleepRevealedbyaNovelSystemsGeneticAnalysisoftheMicrobiomeinCollaborativeCrossMiceGeneticsofComplexTraits" class="collapse"><p>The microbiome influences health and disease through complex networks of host genetics, genomics, microbes, and environment. Identifying the mechanisms of these interactions has remained challenging. Systems genetics in laboratory mice (<I>Mus musculus</I>) enables data-driven discovery of biological network components and mechanisms of host&ndash;microbial interactions underlying disease phenotypes. To examine the interplay among the whole host genome, transcriptome, and microbiome, we mapped QTL and correlated the abundance of cecal messenger RNA, luminal microflora, physiology, and behavior in a highly diverse Collaborative Cross breeding population. One such relationship, regulated by a variant on chromosome 7, was the association of <I>Odoribacter</I> (Bacteroidales) abundance and sleep phenotypes. In a test of this association in the BKS.Cg-<I>Dock7<sup>m</sup></I> +/+ <I>Lepr<sup>db</sup>/</I>J mouse model of obesity and diabetes, known to have abnormal sleep and colonization by <I>Odoribacter</I>, treatment with antibiotics altered sleep in a genotype-dependent fashion. The many other relationships extracted from this study can be used to interrogate other diseases, microbes, and mechanisms.</p></div><hr>
<h5><b>[PlantCell]</b> <a href=http://www.plantcell.org/cgi/content/short/32/3/547?rss=1>Diatom Molecular Research Comes of Age: Model Species for Studying Phytoplankton Biology and Diversity</a>. Falciatore, A., Jaubert, M., Bouly, J.-P., Bailleul, B., Mock, T. 2020-03-02T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractDiatomMolecularResearchComesofAgeModelSpeciesforStudyingPhytoplanktonBiologyandDiversity">Abstract</button></h5><div id="abstractDiatomMolecularResearchComesofAgeModelSpeciesforStudyingPhytoplanktonBiologyandDiversity" class="collapse"><p>Diatoms are the world&rsquo;s most diverse group of algae, comprising at least 100,000 species. Contributing ~20% of annual global carbon fixation, they underpin major aquatic food webs and drive global biogeochemical cycles. Over the past two decades, <I>Thalassiosira pseudonana</I> and <I>Phaeodactylum tricornutum</I> have become the most important model systems for diatom molecular research, ranging from cell biology to ecophysiology, due to their rapid growth rates, small genomes, and the cumulative wealth of associated genetic resources. To explore the evolutionary divergence of diatoms, additional model species are emerging, such as <I>Fragilariopsis cylindrus</I> and <I>Pseudo-nitzschia multistriata</I>. Here, we describe how functional genomics and reverse genetics have contributed to our understanding of this important class of microalgae in the context of evolution, cell biology, and metabolic adaptations. Our review will also highlight promising areas of investigation into the diversity of these photosynthetic organisms, including the discovery of new molecular pathways governing the life of secondary plastid-bearing organisms in aquatic environments.</p></div>
<h5><b>[PlantCell]</b> <a href=http://www.plantcell.org/cgi/content/short/32/3/650?rss=1>Genome Evolution in Arabideae Was Marked by Frequent Centromere Repositioning</a>. Mandakova, T., Hlouskova, P., Koch, M. A., Lysak, M. A. 2020-03-02T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractGenomeEvolutioninArabideaeWasMarkedbyFrequentCentromereRepositioning">Abstract</button></h5><div id="abstractGenomeEvolutioninArabideaeWasMarkedbyFrequentCentromereRepositioning" class="collapse"><p>Centromere position may change despite conserved chromosomal collinearity. Centromere repositioning and evolutionary new centromeres (ENCs) were frequently encountered during vertebrate genome evolution but only rarely observed in plants. The largest crucifer tribe, Arabideae (~550 species; Brassicaceae, the mustard family), diversified into several well-defined subclades in the virtual absence of chromosome number variation. Bacterial artificial chromosome&ndash;based comparative chromosome painting uncovered a constancy of genome structures among 10 analyzed genomes representing seven Arabideae subclades classified as four genera: <I>Arabis</I>, <I>Aubrieta</I>, <I>Draba</I>, and <I>Pseudoturritis</I>. Interestingly, the intra-tribal diversification was marked by a high frequency of ENCs on five of the eight homoeologous chromosomes in the crown-group genera, but not in the most ancestral <I>Pseudoturritis</I> genome. From the 32 documented ENCs, at least 26 originated independently, including 4 ENCs recurrently formed at the same position in not closely related species. While chromosomal localization of ENCs does not reflect the phylogenetic position of the Arabideae subclades, centromere seeding was usually confined to long chromosome arms, transforming acrocentric chromosomes to (sub)metacentric chromosomes. Centromere repositioning is proposed as the key mechanism differentiating overall conserved homoeologous chromosomes across the crown-group Arabideae subclades. The evolutionary significance of centromere repositioning is discussed in the context of possible adaptive effects on recombination and epigenetic regulation of gene expression.</p></div><hr>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1222?rss=1>The Arabidopsis Protein CGL20 Is Required for Plastid 50S Ribosome Biogenesis</a>. Reiter, B., Vamvaka, E., Marino, G., Kleine, T., J..hns, P., Bolle, C., Leister, D., Ru&#x0308;hle, T. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTheArabidopsisProteinCGL20IsRequiredforPlastid50SRibosomeBiogenesis">Abstract</button></h5><div id="abstractTheArabidopsisProteinCGL20IsRequiredforPlastid50SRibosomeBiogenesis" class="collapse"><p>Biogenesis of plastid ribosomes is facilitated by auxiliary factors that process and modify ribosomal RNAs (rRNAs) or are involved in ribosome assembly. In comparison with their bacterial and mitochondrial counterparts, the biogenesis of plastid ribosomes is less well understood, and few auxiliary factors have been described so far. In this study, we report the functional characterization of CONSERVED ONLY IN THE GREEN LINEAGE20 (CGL20) in Arabidopsis (<I>Arabidopsis thaliana</I>; AtCGL20), which is a Pro-rich, ~10-kD protein that is targeted to mitochondria and chloroplasts. In Arabidopsis, CGL20 is encoded by segmentally duplicated genes of high sequence similarity (<I>AtCGL20A</I> and <I>AtCGL20B</I>). Inactivation of these genes in the <I>atcgl20ab</I> mutant led to a visible virescent phenotype and growth arrest at low temperature. The chloroplast proteome, pigment composition, and photosynthetic performance were significantly affected in <I>atcgl20ab</I> mutants. Loss of AtCGL20 impaired plastid translation, perturbing the formation of a hidden break in the 23S rRNA and causing abnormal accumulation of 50S ribosomal subunits in the high-molecular-mass fraction of chloroplast stromal extracts. Moreover, AtCGL20A-eGFP fusion proteins comigrated with 50S ribosomal subunits in Suc density gradients, even after RNase treatment of stromal extracts. Therefore, we propose that AtCGL20 participates in the late stages of the biogenesis of 50S ribosomal subunits in plastids, a role that presumably evolved in the green lineage as a consequence of structural divergence of plastid ribosomes.</p></div>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1297?rss=1>Subdivision of Light Signaling Networks Contributes to Partitioning of C4 Photosynthesis</a>. Hendron, R.-W., Kelly, S. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractSubdivisionofLightSignalingNetworksContributestoPartitioningofC4Photosynthesis">Abstract</button></h5><div id="abstractSubdivisionofLightSignalingNetworksContributestoPartitioningofC4Photosynthesis" class="collapse"><p>Plants coordinate the expression of photosynthesis-related genes in response to growth and environmental changes. In species that conduct two-cell C<SUB>4</SUB> photosynthesis, expression of photosynthesis genes is partitioned such that leaf mesophyll and bundle sheath cells accumulate different components of the photosynthetic pathway. The identities of the regulatory networks that facilitate this partitioning are unknown. Here, we show that differences in light perception between mesophyll and bundle sheath cells facilitate differential regulation and accumulation of photosynthesis gene transcripts in the C<SUB>4</SUB> crop maize (<I>Zea mays</I>). Key components of the photosynthesis gene regulatory network differentially accumulated between mesophyll and bundle sheath cells, indicative of differential network activity across cell types. We further show that blue (but not red) light is necessary and sufficient to activate photosystem II assembly in mesophyll cells in etiolated maize. Finally, we demonstrate that 61% of all light-induced mesophyll and bundle sheath genes were induced only by blue light or only by red light, but not both. These findings provide evidence that subdivision of light signaling networks is a component of cellular partitioning of C<SUB>4</SUB> photosynthesis in maize.</p></div>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1359?rss=1>Analysis of Soybean Long Non-Coding RNAs Reveals a Subset of Small Peptide-Coding Transcripts</a>. Lin, X., Lin, W., Ku, Y.-S., Wong, F.-L., Li, M.-W., Lam, H.-M., Ngai, S.-M., Chan, T.-F. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractAnalysisofSoybeanLongNonCodingRNAsRevealsaSubsetofSmallPeptideCodingTranscripts">Abstract</button></h5><div id="abstractAnalysisofSoybeanLongNonCodingRNAsRevealsaSubsetofSmallPeptideCodingTranscripts" class="collapse"><p>Long non-coding RNAs (lncRNAs) are defined as non-protein&ndash;coding transcripts that are at least 200 nucleotides long. They are known to play pivotal roles in regulating gene expression, especially during stress responses in plants. We used a large collection of in-house transcriptome data from various soybean (<I>Glycine max</I> and <I>Glycine soja</I>) tissues treated under different conditions to perform a comprehensive identification of soybean lncRNAs. We also retrieved publicly available soybean transcriptome data that were of sufficient quality and sequencing depth to enrich our analysis. In total, RNA-sequencing data of 332 samples were used for this analysis. An integrated reference-based, de novo transcript assembly was developed that identified ~69,000 lncRNA gene loci. We showed that lncRNAs are distinct from both protein-coding transcripts and genomic background noise in terms of length, number of exons, transposable element composition, and sequence conservation level across legume species. The tissue-specific and time-specific transcriptional responses of the lncRNA genes under some stress conditions may suggest their biological relevance. The transcription start sites of lncRNA gene loci tend to be close to their nearest protein-coding genes, and they may be transcriptionally related to the protein-coding genes, particularly for antisense and intronic lncRNAs. A previously unreported subset of small peptide-coding transcripts was identified from these lncRNA loci via tandem mass spectrometry, which paved the way for investigating their functional roles. Our results also highlight the present inadequacy of the bioinformatic definition of lncRNA, which excludes those lncRNA gene loci with small open reading frames from being regarded as protein-coding.</p></div>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1440?rss=1>TaANR1-TaBG1 and TaWabi5-TaNRT2s/NARs Link ABA Metabolism and Nitrate Acquisition in Wheat Roots</a>. Wang, M., Zhang, P., Liu, Q., Li, G., Di, D., Xia, G., Kronzucker, H. J., Fang, S., Chu, J., Shi, W. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractTaANR1TaBG1andTaWabi5TaNRT2sNARsLinkABAMetabolismandNitrateAcquisitioninWheatRoots">Abstract</button></h5><div id="abstractTaANR1TaBG1andTaWabi5TaNRT2sNARsLinkABAMetabolismandNitrateAcquisitioninWheatRoots" class="collapse"><p>Nitrate is the preferred form of nitrogen for most plants, acting both as a nutrient and a signaling molecule. However, the components and regulatory factors governing nitrate uptake in bread wheat (<I>Triticum aestivum</I>), one of the world&rsquo;s most important crop species, have remained unclear, largely due to the complexity of its hexaploid genome. Here, based on recently released whole-genome information for bread wheat, the high-affinity nitrate transporter2 (<I>NRT2</I>) and the nitrate-assimilation-related (<I>NAR</I>) gene family are characterized. We show that abscisic acid (ABA)- Glc ester deconjugation is stimulated in bread wheat roots by nitrate resupply following nitrate withdrawal, leading to enhanced root-tissue ABA accumulation, and that this enhancement, in turn, affects the expression of root-type <I>NRT2</I>/<I>NAR</I> genes. TaANR1 is shown to regulate nitrate-mediated ABA accumulation by directly activating <I>TaBG1</I>, while TaWabi5 is involved in ABA-mediated NO<SUB>3</SUB><sup>&ndash;</sup> induction of <I>NRT2</I>/<I>NAR</I> genes. Building on previous evidence establishing ABA involvement in the developmental response to high-nitrate stress, our study suggests that ABA also contributes to the optimization of nitrate uptake by regulating the expression of <I>NRT2</I>/<I>NAR</I> genes under limited nitrate supply, offering a new target for improvement of nitrate absorption in crops.</p></div>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1494?rss=1>Low-Phosphate Chromatin Dynamics Predict a Cell Wall Remodeling Network in Rice Shoots</a>. Foroozani, M., Zahraeifard, S., Oh, D.-H., Wang, G., Dassanayake, M., Smith, A. P. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractLowPhosphateChromatinDynamicsPredictaCellWallRemodelingNetworkinRiceShoots">Abstract</button></h5><div id="abstractLowPhosphateChromatinDynamicsPredictaCellWallRemodelingNetworkinRiceShoots" class="collapse"><p>Phosphorus (P) is an essential plant macronutrient vital to fundamental metabolic processes. Plant-available P is low in most soils, making it a frequent limiter of growth. Declining P reserves for fertilizer production exacerbates this agricultural challenge. Plants modulate complex responses to fluctuating P levels via global transcriptional regulatory networks. Although chromatin structure plays a substantial role in controlling gene expression, the chromatin dynamics involved in regulating P homeostasis have not been determined. Here we define distinct chromatin states across the rice (<I>Oryza sativa</I>) genome by integrating multiple chromatin marks, including the H2A.Z histone variant, H3K4me3 modification, and nucleosome positioning. In response to P starvation, 40% of all protein-coding genes exhibit a transition from one chromatin state to another at their transcription start site. Several of these transitions are enriched in subsets of genes differentially expressed under P deficiency. The most prominent subset supports the presence of a coordinated signaling network that targets cell wall structure and is regulated in part via a decrease of H3K4me3 at transcription start sites. The P starvation-induced chromatin dynamics and correlated genes identified here will aid in enhancing P use efficiency in crop plants, benefitting global agriculture.</p></div>
<h5><b>[PlantPhysiol]</b> <a href=http://www.plantphysiol.org/cgi/content/short/182/3/1510?rss=1>Full-Length Transcript-Based Proteogenomics of Rice Improves Its Genome and Proteome Annotation</a>. Chen, M.-X., Zhu, F.-Y., Gao, B., Ma, K.-L., Zhang..an, Y., Zhang, D., Xiao, S., Zhang, J., Liu, Y.-G. 2020-03-03T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractFullLengthTranscriptBasedProteogenomicsofRiceImprovesItsGenomeandProteomeAnnotation">Abstract</button></h5><div id="abstractFullLengthTranscriptBasedProteogenomicsofRiceImprovesItsGenomeandProteomeAnnotation" class="collapse"><p>Rice (<I>Oryza sativa</I>) molecular breeding has gained considerable attention in recent years, but inaccurate genome annotation hampers its progress and functional studies of the rice genome. In this study, we applied single-molecule long-read RNA sequencing (lrRNA_seq)-based proteogenomics to reveal the complexity of the rice transcriptome and its coding abilities. Surprisingly, approximately 60% of loci identified by lrRNA_seq are associated with natural antisense transcripts (NATs). The high-density genomic arrangement of NAT genes suggests their potential roles in the multifaceted control of gene expression. In addition, a large number of fusion and intergenic transcripts have been observed. Furthermore, 906,456 transcript isoforms were identified, and 72.9% of the genes can generate splicing isoforms. A total of 706,075 posttranscriptional events were subsequently categorized into 10 subtypes, demonstrating the interdependence of posttranscriptional mechanisms that contribute to transcriptome diversity. Parallel short-read RNA sequencing indicated that lrRNA_seq has a superior capacity for the identification of longer transcripts. In addition, over 190,000 unique peptides belonging to 9,706 proteoforms/protein groups were identified, expanding the diversity of the rice proteome. Our findings indicate that the genome organization, transcriptome diversity, and coding potential of the rice transcriptome are far more complex than previously anticipated.</p></div><hr>
<h5><b>[NatMethods]</b> <a href=https://www.nature.com/articles/s41592-020-0775-2>3D ATAC-PALM: super-resolution imaging of the accessible genome</a>. Zhe Liu 2020-03-16 <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstract3DATACPALMsuperresolutionimagingoftheaccessiblegenome">Abstract</button></h5><div id="abstract3DATACPALMsuperresolutionimagingoftheaccessiblegenome" class="collapse"><p>Nature Methods, Published online: 16 March 2020; <a href="https://www.nature.com/articles/s41592-020-0775-2">doi:10.1038/s41592-020-0775-2</a></p>3D ATAC-PALM integrates ATAC with super-resolution imaging for nanoscale views of the accessible genome. When combined with FISH, protein fluorescence and genetic perturbation, the method enables investigation of accessible chromatin in situ.</div><hr>
<h5><b>[Bioinformatics]</b> <a href=https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa197/5809526?rss=1>powmic: an R package for power assessment in microbiome case-control studies</a>. Chen L, Birol I. Wed, 18 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractpowmicanRpackageforpowerassessmentinmicrobiomecasecontrolstudies">Abstract</button></h5><div id="abstractpowmicanRpackageforpowerassessmentinmicrobiomecasecontrolstudies" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div><div class="boxTitle">Summary</div>Power analysis is essential to decide the sample size of metagenomic sequencing experiments in a case-control study for identifying differentially abundant microbes. However, the complexity of microbial data characteristics such as excessive zeros, over-dispersion, compositionality, intrinsically microbial correlations and variable sequencing depths makes the power analysis particularly challenging because the analytical form is usually unavailable. Here, we develop a simulation-based power assessment strategy and R package powmic, which considers the complexity of microbial data characteristics. A real data example demonstrates the usage of powmic.<div class="boxTitle">Availability and Implementation</div>powmic R package and online tutorial are available at <a href="https://github.com/lichen-lab/powmic">https://github.com/lichen-lab/powmic</a><div class="boxTitle">Supplementary Information</div>Supplementary dataSupplementary data are available at Bioinformatics online.</span></div>
<h5><b>[Bioinformatics]</b> <a href=https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa191/5809142?rss=1>A Blind and Independent Benchmark Study for Detecting Differentially Methylated Regions in Plants</a>. Kreutz C, Can N, Bruening R, et al. Tue, 17 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractABlindandIndependentBenchmarkStudyforDetectingDifferentiallyMethylatedRegionsinPlants">Abstract</button></h5><div id="abstractABlindandIndependentBenchmarkStudyforDetectingDifferentiallyMethylatedRegionsinPlants" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div><div class="boxTitle">Motivation</div>Bisulfite sequencing (BS-seq) is a state-of-the-art technique for investigating methylation of the DNA to gain insights into the epigenetic regulation. Several algorithms have been published for identification of differentially methylated regions (DMRs). However, the performances of the individual methods remain unclear and it is difficult to optimally select an algorithm in application settings.<div class="boxTitle">Results</div>We analyzed BS-seq data from four plants covering three taxonomic groups. We first characterized the data using multiple summary statistics describing methylation levels, coverage, noise, as well as frequencies, magnitudes and lengths of methylated regions. Then, simulated data sets with most similar characteristics to real experimental data were created. Seven different algorithms (<span style="font-style:italic;">metilene</span>, <span style="font-style:italic;">methylKit</span>, <span style="font-style:italic;">MOABS</span>, <span style="font-style:italic;">DMRcate</span>, <span style="font-style:italic;">Defiant</span>, <span style="font-style:italic;">BSmooth</span>, <span style="font-style:italic;">MethylSig</span>) for DMR identification were applied and their performances were assessed. A blind and independent study design was chosen to reduce bias and to derive practical method selection guidelines. Overall, <span style="font-style:italic;">metilene</span> had superior performance in most settings. Data attributes such as coverage and spread of the DMR lengths were found to be useful for selecting the best method for DMR detection. A decision tree to select the optimal approach based on these data attributes is provided. The presented procedure might serve as a general strategy for deriving algorithm selection rules tailored to demands in specific application settings.<div class="boxTitle">Availability</div>Scripts that were used for the analyses and that can be used for prediction of the optimal algorithm are provided at <a href="https://github.com/kreutz-lab/DMR-DecisionTree">https://github.com/kreutz-lab/DMR-DecisionTree</a>. Simulated and experimental data are available at https://doi.org/10.6084/m9.figshare.11619045<strong>Supplementary Information</strong> is available at <span style="font-style:italic;">Bioinformatics</span> online.</span></div>
<h5><b>[Bioinformatics]</b> <a href=https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa192/5809141?rss=1>LogoJS: a Javascript package for creating sequence logos and embedding them in web applications</a>. Pratt H, Weng Z, Valencia A. Tue, 17 Mar 2020  <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractLogoJSaJavascriptpackageforcreatingsequencelogosandembeddingtheminwebapplications">Abstract</button></h5><div id="abstractLogoJSaJavascriptpackageforcreatingsequencelogosandembeddingtheminwebapplications" class="collapse"><span class="paragraphSection"><div class="boxTitle">Abstract</div><div class="boxTitle">Summary</div>Sequence logos were introduced nearly 30 years ago as a human-readable format for representing consensus sequences, and they remain widely used. As new experimental and computational techniques have developed, logos have been extended: extra symbols represent covalent modifications to nucleotides, logos with multiple letters at each position illustrate models with multi-nucleotide features, and symbols extending below the x-axis may represent a binding energy penalty for a residue or a negative weight output from a neural network. Web-based visualization tools for genomic data are increasingly taking advantage of modern web technology to offer dynamic, interactive figures to users, but support for sequence logos remains limited. Here we present LogoJS, a Javascript package for rendering customizable, interactive, vector-graphic sequence logos and embedding them in web applications. LogoJS supports all the aforementioned logo extensions and is bundled with a companion web application for creating and sharing logos.<div class="boxTitle">Availability</div>LogoJS is implemented both in plain Javascript and ReactJS, a popular user-interface framework. The web application is hosted at <a href="http://logojs.wenglab.org">logojs.wenglab.org</a>. All major browsers and operating systems are supported. The package and application are open-source; code is available at GitHub.<div class="boxTitle">Supplementary information</div>Supplementary dataSupplementary data are available at <span style="font-style:italic;">Bioinformatics</span> online.</span></div><hr>
<h5><b>[PLoSComputBiol]</b> <a href=http://feeds.plos.org/~r/ploscompbiol/NewArticles/~3/JDlGFJP7808/article>PPanGGOLiN: Depicting microbial diversity via a partitioned pangenome graph</a>. David Vallenet 2020-03-19T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractPPanGGOLiNDepictingmicrobialdiversityviaapartitionedpangenomegraph">Abstract</button></h5><div id="abstractPPanGGOLiNDepictingmicrobialdiversityviaapartitionedpangenomegraph" class="collapse"><p>by Guillaume Gautreau, Adelme Bazin, Mathieu Gachet, Rémi Planel, Laura Burlot, Mathieu Dubois, Amandine Perrin, Claudine Médigue, Alexandra Calteau, Stéphane Cruveiller, Catherine Matias, Christophe Ambroise, Eduardo P. C. Rocha, David Vallenet</p>The use of comparative genomics for functional, evolutionary, and epidemiological studies requires methods to classify gene families in terms of occurrence in a given species. These methods usually lack multivariate statistical models to infer the partitions and the optimal number of classes and don’t account for genome organization. We introduce a graph structure to model pangenomes in which nodes represent gene families and edges represent genomic neighborhood. Our method, named PPanGGOLiN, partitions nodes using an Expectation-Maximization algorithm based on multivariate Bernoulli Mixture Model coupled with a Markov Random Field. This approach takes into account the topology of the graph and the presence/absence of genes in pangenomes to classify gene families into persistent, cloud, and one or several shell partitions. By analyzing the partitioned pangenome graphs of isolate genomes from 439 species and metagenome-assembled genomes from 78 species, we demonstrate that our method is effective in estimating the persistent genome. Interestingly, it shows that the shell genome is a key element to understand genome dynamics, presumably because it reflects how genes present at intermediate frequencies drive adaptation of species, and its proportion in genomes is independent of genome size. The graph-based approach proposed by PPanGGOLiN is useful to depict the overall genomic diversity of thousands of strains in a compact structure and provides an effective basis for very large scale comparative genomics. The software is freely available at https://github.com/labgem/PPanGGOLiN.</div>
<h5><b>[PLoSComputBiol]</b> <a href=http://feeds.plos.org/~r/ploscompbiol/NewArticles/~3/4tX46Deg1pQ/article>CAncer bioMarker Prediction Pipeline (CAMPP)—A standardized framework for the analysis of quantitative biological data</a>. Elena Papaleo 2020-03-16T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractCAncerbioMarkerPredictionPipelineCAMPPAstandardizedframeworkfortheanalysisofquantitativebiologicaldata">Abstract</button></h5><div id="abstractCAncerbioMarkerPredictionPipelineCAMPPAstandardizedframeworkfortheanalysisofquantitativebiologicaldata" class="collapse"><p>by Thilde Terkelsen, Anders Krogh, Elena Papaleo</p>With the improvement of -omics and next-generation sequencing (NGS) methodologies, along with the lowered cost of generating these types of data, the analysis of high-throughput biological data has become standard both for forming and testing biomedical hypotheses. Our knowledge of how to normalize datasets to remove latent undesirable variances has grown extensively, making for standardized data that are easily compared between studies. Here we present the CAncer bioMarker Prediction Pipeline (CAMPP), an open-source R-based wrapper (https://github.com/ELELAB/CAncer-bioMarker-Prediction-Pipeline -CAMPP) intended to aid bioinformatic software-users with data analyses. CAMPP is called from a terminal command line and is supported by a user-friendly manual. The pipeline may be run on a local computer and requires little or no knowledge of programming. To avoid issues relating to R-package updates, a renv .lock file is provided to ensure R-package stability. Data-management includes missing value imputation, data normalization, and distributional checks. CAMPP performs (I) k-means clustering, (II) differential expression/abundance analysis, (III) elastic-net regression, (IV) correlation and co-expression network analyses, (V) survival analysis, and (VI) protein-protein/miRNA-gene interaction networks. The pipeline returns tabular files and graphical representations of the results. We hope that CAMPP will assist in streamlining bioinformatic analysis of quantitative biological data, whilst ensuring an appropriate bio-statistical framework.</div>
<h5><b>[PLoSComputBiol]</b> <a href=http://feeds.plos.org/~r/ploscompbiol/NewArticles/~3/AEqpGdCC3f8/article>Comprehensive analysis of structural and sequencing data reveals almost unconstrained chain pairing in TCRαβ complex</a>. Mikhail Shugay 2020-03-12T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractComprehensiveanalysisofstructuralandsequencingdatarevealsalmostunconstrainedchainpairinginTCRαβcomplex">Abstract</button></h5><div id="abstractComprehensiveanalysisofstructuralandsequencingdatarevealsalmostunconstrainedchainpairinginTCRαβcomplex" class="collapse"><p>by Dmitrii S. Shcherbinin, Vlad A. Belousov, Mikhail Shugay</p>Antigen recognition by T-cells is guided by the T-cell receptor (TCR) heterodimer formed by α and β chains. A huge diversity of TCR sequences should be maintained by the immune system in order to be able to mount an effective response towards foreign pathogens, so, due to cooperative binding of α and β chains to the pathogen, any constraints on chain pairing can have a profound effect on immune repertoire structure, diversity and antigen specificity. By integrating available structural data and paired chain sequencing results we were able to show that there are almost no constraints on pairing in TCRαβ complexes, allowing naive T-cell repertoire to reach the highest possible diversity. Additional analysis reveals that the specific choice of contacting amino acids can still have a profound effect on complex conformation. Moreover, antigen-driven selection can distort the uniform landscape of chain pairing, while small, yet significant, differences in the pairing can be attributed to various specialized T-cell subsets such as MAIT and iNKT T-cells, as well as other TCR sets specific to certain antigens.</div>
<h5><b>[PLoSComputBiol]</b> <a href=http://feeds.plos.org/~r/ploscompbiol/NewArticles/~3/uTE3EjW-gJw/article>ASA<sup>3</sup>P: An automatic and scalable pipeline for the assembly, annotation and higher-level analysis of closely related bacterial isolates</a>. Alexander Goesmann 2020-03-05T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractASAsup3supPAnautomaticandscalablepipelinefortheassemblyannotationandhigherlevelanalysisofcloselyrelatedbacterialisolates">Abstract</button></h5><div id="abstractASAsup3supPAnautomaticandscalablepipelinefortheassemblyannotationandhigherlevelanalysisofcloselyrelatedbacterialisolates" class="collapse"><p>by Oliver Schwengers, Andreas Hoek, Moritz Fritzenwanker, Linda Falgenhauer, Torsten Hain, Trinad Chakraborty, Alexander Goesmann</p>Whole genome sequencing of bacteria has become daily routine in many fields. Advances in DNA sequencing technologies and continuously dropping costs have resulted in a tremendous increase in the amounts of available sequence data. However, comprehensive in-depth analysis of the resulting data remains an arduous and time-consuming task. In order to keep pace with these promising but challenging developments and to transform raw data into valuable information, standardized analyses and scalable software tools are needed. Here, we introduce ASA<sup>3</sup>P, a fully automatic, locally executable and scalable assembly, annotation and analysis pipeline for bacterial genomes. The pipeline automatically executes necessary data processing steps, <i>i</i>.<i>e</i>. quality clipping and assembly of raw sequencing reads, scaffolding of contigs and annotation of the resulting genome sequences. Furthermore, ASA<sup>3</sup>P conducts comprehensive genome characterizations and analyses, <i>e</i>.<i>g</i>. taxonomic classification, detection of antibiotic resistance genes and identification of virulence factors. All results are presented via an HTML5 user interface providing aggregated information, interactive visualizations and access to intermediate results in standard bioinformatics file formats. We distribute ASA<sup>3</sup>P in two versions: a locally executable Docker container for small-to-medium-scale projects and an OpenStack based cloud computing version able to automatically create and manage self-scaling compute clusters. Thus, automatic and standardized analysis of hundreds of bacterial genomes becomes feasible within hours. The software and further information is available at: asap.computational.bio.</div><hr>
<h5><b>[BMCBioinformatics]</b> <a href=https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3428-7>LCQS: an efficient lossless compression tool of quality scores with random access functionality</a>. Jiabing Fu, Bixin Ke and Shoubin Dong 2020-03-18T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractLCQSanefficientlosslesscompressiontoolofqualityscoreswithrandomaccessfunctionality">Abstract</button></h5><div id="abstractLCQSanefficientlosslesscompressiontoolofqualityscoreswithrandomaccessfunctionality" class="collapse">Advanced sequencing machines dramatically speed up the generation of genomic data, which makes the demand of efficient compression of sequencing data extremely urgent and significant. As the most difficult par...</div>
<h5><b>[BMCBioinformatics]</b> <a href=https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3433-x>RASflow: an RNA-Seq analysis workflow with Snakemake</a>. Xiaokang Zhang and Inge Jonassen 2020-03-18T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractRASflowanRNASeqanalysisworkflowwithSnakemake">Abstract</button></h5><div id="abstractRASflowanRNASeqanalysisworkflowwithSnakemake" class="collapse">With the cost of DNA sequencing decreasing, increasing amounts of RNA-Seq data are being generated giving novel insight into gene expression and regulation. Prior to analysis of gene expression, the RNA-Seq da...</div>
<h5><b>[BMCBioinformatics]</b> <a href=https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3434-9>SEQing: web-based visualization of iCLIP and RNA-seq data in an interactive python framework</a>. Martin Lewinski, Yannik Bramkamp, Tino Köster and Dorothee Staiger 2020-03-18T <button type="button" class="btn btn-primary btn-xs" data-toggle="collapse" data-target="#abstractSEQingwebbasedvisualizationofiCLIPandRNAseqdatainaninteractivepythonframework">Abstract</button></h5><div id="abstractSEQingwebbasedvisualizationofiCLIPandRNAseqdatainaninteractivepythonframework" class="collapse">RNA-binding proteins interact with their target RNAs at specific sites. These binding sites can be determined genome-wide through individual nucleotide resolution crosslinking immunoprecipitation (iCLIP). Subs...</div>
<a href="archive/rss-03-20-2020.html" class="btn btn-primary btn-xs" role="button">Previous Week</a>
<a href="./rss-all.html" class="btn btn-primary btn-xs" role="button">Full Repertoire</a>
</div>
</body>
</html>
